---

title: Crystal structure of the complex of hepatocyte growth factor beta chain with met receptor and methods of use
abstract: The disclosure provides a crystal structure of a complex of the HGF Î²-chain with am extracellular fragment of the Met receptor, as well as use of the crystal structure in the design, identification, and selection of ligands that modulate the Met Receptor and the interaction of HGF with the Met receptor.
url: http://patft.uspto.gov/netacgi/nph-Parser?Sect1=PTO2&Sect2=HITOFF&p=1&u=%2Fnetahtml%2FPTO%2Fsearch-adv.htm&r=1&f=G&l=50&d=PALL&S1=07754458&OS=07754458&RS=07754458
owner: Genentech, Inc.
number: 07754458
owner_city: South San Francisco
owner_country: US
publication_date: 20080117
---
This application is a continuation application of U.S. patent application Ser. No. 11 124 607 filed May 6 2005 which claims the benefit under 35 U.S.C. 119 e to U.S. Provisional Application Ser. No. 60 568 865 filed May 6 2004 which applications are hereby incorporated by reference in their entirety.

The receptor tyrosine kinase Met and its ligand hepatocyte growth factor HGF also called scatter factor have been implicated in promoting invasive growth of many tumor types due to inappropriate activation of Met function Jankowski et al. 2003 Nardone et al. 2003 Trusolino and Comoglio 2002 Birchmeier et al. 2003 . This activation can arise from a variety of sources but in each case the Met receptor activates signaling cascades that normally function to organize groups of cells into branching tubular structures that are present in a variety of organs Montesano et al. 1992 Sonnenberg et al. 1993 Rosen et al. 1994 Trusolino and Comoglio 2002 Zhang and Vande Woude 2003 . The Met receptor plays a unique role during development as a master switch which can stimulate proliferation and motility necessary for the full program of growth and scattering of cells. Its role in the invasiveness of many cancers makes it an attractive target for therapeutics Ma et al. 2003 . However many questions remain about how the ligand HGF binds to Met and induces its tyrosine kinase cascade and thus leads to a biological response.

The Met receptor is part of a larger family of growth factor receptors with identical domain architecture that includes the Ron and Sea receptors Monsin et al. 1992 Huff et al. 1993 . The extracellular portions of Met family members are composed of three domain types. The N terminal 500 residues fold into a Sema domain which shares sequence homology with domains found in the Semaphorin and plexin families of neural development proteins Winburg et al. 1998 . As reported recently Sema domains form a 7 bladed propeller structure Antipenkov et al. 2003 Love et al. 2003 . Met undergoes proteolytic cleavage within the Sema domain during normal processing although the role for this remains unclear since cells that are unable to cleave Met show normal levels of Met activation upon ligand binding Komada et al. 1993 . A PSI domain a small domain spanning about 50 residues and containing 4 disulfide bonds follows the Sema domain. In addition to the Met receptor family PSI domains are also found in the plexins Semaphorins and integrins hence its name Bork et al. 1999 . In Met the PSI domain is connected via 4 IPT domains to the transmembrane helix and the kinase domain in the intracellular portion of the receptor. IPT domains are related to immunoglobulin like domains and are named after their presence in plexins and transcription factors Takagi et al. 1995 .

HGF is a large growth factor of 728 residues that is produced as an inactive single chain precursor which is proteolytically processed to form the biologically active disulfide linked heterodimer Nakamura et al. 1989 Hartmann et al. 1992 Kataoka et al. 2003 . The chain folds into an N terminal domain N domain followed by 4 Kringle domains. The chain starts with residue Val495 and is homologous to the protease domain of chymotrypsin like serine proteases which like HGF are activated by a proteolytic cleavage event Perona and Craig 1995 Hedstrom 2002 . However no protease activity has been demonstrated for HGF chain Lokker et al. 1992 consistent with the absence of the key serine and histidine residues that are part of the catalytic triad Asp c102 His c57 Ser c195 chymotrypsinogen numbering required for catalytic activity in serine proteases.

Comparisons of the biologically active two chain HGF and the inactive single chain HGF precursor on the Met receptor have shown that both forms of HGF bind Met with similar affinity but only the cleaved mature form of HGF is able to activate Met Lokker et al. 1992 . In addition various C terminally truncated fragments of the chain termed NK1 NK2 or NK4 depending on the number of Kringle domains retained bind Met in many cases they act as potent antagonists of Met receptor function Cioce et al. 1996 Chan et al. 1991 Date et al. 1997 . Studies involving the cross linking of Met receptors by a variety of specific antibodies to its extracellular domain have demonstrated that simple dimerization of Met is sufficient for activation Prat et al. 1998 . Based on these characteristics the fundamental mechanism for Met dimerization remains unclear.

Currently there is no detailed structural information about HGF chain complexed with Met receptor. A completely solved crystal structure of the HGF chain complexed with Met receptor is needed for example for assays for Met ligand e.g. HGF chain interaction and function modeling the structure function relationship of Met and other molecules diagnostic assays for mutation induced pathologies and rational design of agents useful in modulating Met or HGF activity or activation.

In some embodiments the present disclosure provides a crystalline form of hepatocyte growth factor beta chain HGF complexed with Met receptor and the structural coordinates of the crystal. Coordinates of a crystal structure solved by molecular replacement are listed in Table 2. In some embodiments HGF comprises an amino acid sequence of SEQ ID NO 1 or conservative substitutions thereof and the Met receptor comprises an amino acid sequence of SEQ ID NO 3 or conservative substitutions thereof.

In some embodiments the disclosure provides a crystal structure of HGF complexed with Met receptor as well as use of the crystal structure to model Met receptor activity when complexed with HGF . This use of the structure includes modeling the interaction of ligands with the Met receptor activation and inhibition of Met receptor and the rational design of modulators of Met receptor activity. For example these modulators include ligands that interact with Met receptor and modulate Met receptor activities such as cell migration HGF binding to Met and Met phosphorylation and signaling.

In other embodiments the amino acid residues that form the binding site for the Met receptor on HGF are identified and are useful for example in methods to model the structure of HGF binding site and to identify agents that can bind or fit into the binding site. In addition the amino acid positions that form the binding site for HGF on Met have been identified and are useful for example in methods to model the structure of the Met ligand binding site and to identify other agents that can bind or fit into the binding site.

PSI domain is a small domain which follows the Sema domain of Met and spans about 50 residues and contains 4 disulfide bonds

The term hepatocyte growth factor or HGF as used herein refers unless specifically or contextually indicated otherwise to any native or variant whether native or synthetic HGF polypeptide that is capable of binding to Met and or activating the HGF Met signaling pathway under conditions that permit such process to occur for example conditions that allow for the formation of the two chain form. The term wild type HGF sequence generally refers to an amino acid sequence found in a naturally occurring HGF and includes naturally occurring truncated or secreted forms variant forms e.g. alternatively spliced forms and naturally occurring allelic variants.

 HGF or HGF chain HGF beta or variations thereof refers to any HGF chain having the conformation that is adopted by wild type HGF chain upon conversion of wild type HGF protein from a single chain form to a 2 chain form i.e. and chain . In some embodiments the conversion results at least in part from cleavage between residue 494 and residue 495 of the wild type HGF protein. In some embodiments the conformation refers specifically to the conformation of the activation domain of the protease like domain in the chain. In some embodiments the conformation refers even more specifically to the conformation of the active site of the protease like domain in the HGF chain. Generally adoption of the conformation reveals a Met binding site as described herein. HGF includes variants of wild type HGF for example a variant comprising an amino acid sequence of SEQ ID NO 1. The HGF chain may be isolated from a variety of sources such as human tissue or prepared by recombinant or synthetic methods. One embodiment of HGF chain comprises an amino acid sequence of SEQ ID NO 1 in Table 4. Another embodiment of HGF chain comprises an amino acid sequence of SEQ ID NO 14 in Table 8.

 HGF variant as used herein refers to polypeptide that has a different sequence than a reference polypeptide. In some embodiments the reference polypeptide is a HGF polypeptide comprising SEQ ID NO 1 in Table 4. In some embodiments a variant has at least 80 amino acid sequence identity with the HGF amino acid sequence of Table 4 SEQ ID NO 1 or Table 8 SEQ ID NO 14 . The variants include those polypeptides that have substitutions additions or deletions. The variants also include those polypeptides that have at least one conservative amino acid substitutions preferably all of the substitutions are conservative. In some embodiments the HGF variant has about 1 25 conservative amino acid substitutions more preferably about 1 20 conservative amino acids substitutions more preferably about 1 10 conservative amino acid substitutions more preferably about 1 5 conservative amino acid substitutions and more preferably about 1 2 conservative amino acid substitutions. In some embodiments the variants have the biological activity of binding to the Met receptor and or activating it. In other embodiments the variant can bind to the Met receptor but not activate it.

Ordinarily a HGF variant polypeptide will have at least 80 sequence identity more preferably will have at least 81 sequence identity more preferably will have at least 82 sequence identity more preferably will have at least 83 sequence identity more preferably will have at least 84 sequence identity more preferably will have at least 85 sequence identity more preferably will have at least 86 sequence identity more preferably will have at least 87 sequence identity more preferably will have at least 88 sequence identity more preferably will have at least 89 sequence identity more preferably will have at least 90 sequence identity more preferably will have at least 91 sequence identity more preferably will have at least 92 sequence identity more preferably will have at least 93 sequence identity more preferably will have at least 94 sequence identity more preferably will have at least 95 sequence identity more preferably will have at least 96 sequence identity more preferably will have at least 96 sequence identity more preferably will have at least 97 sequence identity more preferably will have at least 98 sequence identity more preferably will have at least 99 sequence identity with a HGF polypeptide having an amino acid sequence comprising SEQ ID NO 1 or SEQ ID NO 14.

The term Met receptor or Met as used herein refers to any native or variant whether native or synthetic Met polypeptide that is capable of binding to and or being activated by HGF. The term wild type Met receptor generally refers to a polypeptide comprising an amino acid sequence found in a naturally occurring Met receptor and includes naturally occurring truncated or secreted forms variant forms e.g. alternatively spliced forms and naturally occurring allelic variants. The Met Sema domain comprises the N terminal 500 amino acid residues of a wild type Met receptor. A PSI domain follows the Sema domain and comprises 50 amino acid residues and has 4 disulfide bonds. Following the PSI domain are four IPT domains. IPT domains are related to immunoglobulin like domains. An embodiment of the Met receptor comprises an amino acid sequence of SEQ ID NO 2 as shown in Table 5. An embodiment of the extracellular fragment including the Sema domain of the Met receptor comprises an amino acid sequence of SEQ ID NO 3 as shown in Table 6 or comprises an amino acid sequence of SEQ ID NO 10 as shown in .

The term Met receptor variant as used herein refers to a polypeptide that has a different sequence than a reference polypeptide wherein the reference polypeptide is the Met receptor that comprises an amino acid sequence of SEQ ID NO 2 or the extracellular fragment of the Met receptor that comprises an amino acid sequence of SEQ ID NO 3. Another embodiment of an extracellular fragment of the Met receptor comprises an amino acid sequence of SEQ ID NO 10 as shown in . An extracellular fragment of Met receptor comprising a sequence of SEQ ID NO 3 has amino acid substitutions at positions 304 308 of wild type sequence to insert a thrombin cleavage site. Variants include those polypeptides that have substitutions deletions and or deletions. Variants also include those polypeptides that have at least one conservative amino acid substitution preferably all of the substitutions are conservative. In some embodiments the Met receptor variant has about 1 25 conservative amino acid substitutions more preferably about 1 20 conservative amino acids substitutions more preferably about 1 10 conservative amino acid substitutions more preferably about 1 5 conservative amino acid substitutions and more preferably about 1 2 conservative amino acid substitutions. In some embodiments the variant has the biological activity of binding to HGF but not becoming activated.

Ordinarily a Met receptor variant will have at least 80 sequence identity to a polypeptide having SEQ ID NO 3 In some embodiments Met receptor polypeptide variants have at least 80 sequence identity more preferably 81 sequence identity more preferably 82 sequence identity more preferably 83 sequence identity more preferably 84 sequence identity more preferably 85 sequence identity more preferably 86 sequence identity more preferably 87 sequence identity more preferably 88 sequence identity more preferably 89 sequence identity more preferably 90 sequence identity more preferably 91 sequence identity more preferably 92 sequence identity more preferably 93 sequence identity more preferably 94 sequence identity more preferably 95 sequence identity more preferably 96 sequence identity more preferably 97 sequence identity more preferably 98 sequence identity more preferably 99 sequence identity or greater to a polypeptide having a sequence of SEQ ID NO 2 or SEQ ID NO 3.

The term binding site as used herein refers to a region of a molecule or molecular complex that as a result of its shape distribution of electrostatic charge and or distribution of nonpolar regions favorably associates with a ligand. Thus a binding site may include or consist of features such as cavities surfaces or interfaces between domains. Ligands that may associate with a binding site include but are not limited to cofactors substrates receptors agonists and antagonists. The term binding site includes a functional binding site and or a structural binding site. A structural binding site includes in contact amino acid residues as determined from examination of a three dimensional structure. Contact can be determined using Van der Waals radii of atoms or by proximity sufficient to exclude solvent typically water from the space between the ligand and the molecule or molecular complex. Some of the in contact amino acid residues may not cause any change in a biochemical assay a cell based assay or an in vivo assay used to define a functional binding site but may contribute to the formation of a three dimensional structure. A functional binding site includes amino acid residues that are identified as binding site residues based upon loss or gain of function for example loss of binding to ligand upon mutation of the residue. In some embodiments the amino acid residues of a functional binding site are a subset of the amino acid residues of the structural binding site.

The term HGF structural binding site includes all or a portion of a molecule or molecular complex whose shape is sufficiently similar to at least a portion of a binding site on HGF for Met as to be expected to bind Met or related structural analogs of Met. A structurally equivalent ligand binding site is defined by a root mean square deviation from the structure coordinates of the backbone atoms of the amino acids that make up binding sites in HGF for Met of at most about 0.70 preferably about 0.5 . In some embodiments a structural binding site for the Met receptor on HGF comprises consists essentially of or consists of at least one amino acid residue corresponding to a residue 513 516 533 534 536 537 539 578 619 647 649 656 668 to 670 673 692 to 697 699 702 705 or 707 or mixtures thereof. Numbering of amino acids is that of the native receptor.

The term Met structural binding site includes all or a portion of a molecule whose shape is sufficiently similar to the binding site on Met for HGF to be expected to bind HGF or structural analogs of HGF . A structurally equivalent Met binding site is defined by root mean square deviation from the structure coordinates of the amino acids that make up the binding sites in Met of at most about 0.70 preferably about 0.5 . In some embodiments a structural binding site for HGF on the Met receptor comprises consists essentially of or consists of at least one amino acid residue corresponding to a residue 124 128 148 167 190 192 218 220 to 224 227 229 to 230 286 or 414 or mixtures thereof. Numbering of amino acids is that of the native receptor.

The term a blade of a propeller refers to a structural feature of the Met receptor. A blade is formed by four antiparallel strands with strand A in the center of the blade followed by strands B and C and with strand D forming the outermost strand of the blade. The 7 blades are arranged in a circular fashion with the N terminal strand forming strand D of the last blade. The AB and CD loops of each blade of the Met Sema domain form the flat bottom face of the propeller and the BC and DA loops form the top face of the propeller. In some embodiments each of the blades of the propellor of the human Met receptor comprise the amino acid sequence as identified in .

 Crystal as used herein refers to one form of a solid state of matter in which atoms are arranged in a pattern that repeats periodically in three dimensions typically forming a lattice.

 Complementary or complement as used herein means the fit or relationship between two molecules that permits interaction including for example space charge three dimensional configuration and the like.

The term corresponding or corresponds refers to an amino acid residue or amino acid sequence that is found at the same positions or positions in a sequence when the amino acid position or sequences are aligned with a reference sequence. In some embodiments the reference sequence is the extracellular fragment of the Met receptor comprising a sequence of SEQ ID NO 3. It will be appreciated that when the amino acid position or sequence is aligned with the reference sequence the numbering of the amino acids may differ from that of the reference sequence or a different numbering system may be utilized.

 Heavy atom derivative as used herein means a derivative produced by chemically modifying a crystal with a heavy atom such as Hg Au or a halogen.

 Structural homolog of Met receptor as used herein refers to a protein that contains one or more amino acid substitutions deletions additions or rearrangements with respect to the amino acid sequence of Met receptor but that when folded into its native conformation exhibits or is reasonably expected to exhibit at least a portion of the tertiary three dimensional structure of the Met receptor. In some embodiments a portion of the three dimensional structure refers to structural domains of the Met receptor including the Sema domain PSI domain IPT domains transmembrane domain and or intracellular domain and combinations thereof. For example structurally homologous molecules of Met receptor include Met receptor variants preferably variants with one or more conservative amino acid substitutions. In some embodiments a Met receptor variant has only conservative amino acid substitutions. Homolog tertiary structure can be probed measured or confirmed by known analytic or diagnostic methods for example X ray NMR circular dichroism a panel of monoclonal antibodies that recognize native Met receptor and like techniques. For example structurally homologous molecules can have substitutions deletions or additions of one or more contiguous or noncontiguous amino acids such as a loop or a domain. Structurally homologous molecules also include modified Met receptor molecules that have been chemically or enzymatically derivatized at one or more constituent amino acid including side chain modifications backbone modifications and N and C terminal modifications including acetylation hydroxylation methylation amidation and the attachment of carbohydrate or lipid moieties cofactors and like modifications.

 Ligand as used herein refers to an agent and or compound that associates with a binding site on a molecule for example Met and or HGF binding sites and may be an antagonist or agonist of Met or HGF activity. Ligands include molecules that mimic HGF binding to Met and in some embodiments are not capable of activating HGF Met signalling pathway.

 Compound refers to molecule that associates with the Met or the HGF or a pharmaceutically acceptable salt ester amide prodrug isomer or metabolite thereof. Pharmaceutically acceptable salt refers to a formulation of a compound that does not compromise the biological activity and properties of the compound. Pharmaceutical salts can be obtained by reacting a binding active compound of the disclosure with inorganic or organic acids such as hydrochloric acid hydrobromic acid sulfuric acid nitric acid phosphoric acid methanesulfonic acid ethanesulfonic acid p toluenesulfonic acid salicylic acid and the like. Prodrug refers to an agent that is converted into the parent drug in vivo. Prodrugs are often useful because in some situations they may be easier to administer than the parent drug. They may for instance be bioavailable by oral administration whereas the parent is not. The prodrug may also have improved solubility in pharmaceutical compositions over the parent drug. An example without limitation of a prodrug would be a compound of the present invention which is administered as an ester the prodrug to facilitate transport across a cell membrane where water solubility is detrimental to mobility but which then is metabolically hydrolyzed to the carboxylic acid the active entity once inside the cell where water solubility is beneficial. A further example of a prodrug might be a short peptide polyaminoacid bonded to an acid group wherein the peptide is metabolized to yield the active moiety.

 Molecular complex as used herein refers to a combination of bound substrate or ligand with polypeptide such as HGF bound to Met or a ligand bound to HGF or Met.

 Machine readable data storage medium as used herein means a data storage material encoded with machine readable data wherein a machine programmed with instructions for using such data and is capable of displaying data in the desired format for example a graphical three dimensional representation of molecules or molecular complexes.

 Scalable as used herein means the increasing or decreasing of distances between coordinates configuration of points by a scalar factor while keeping the angles essentially the same.

 Space group symmetry as used herein means the whole symmetry of the crystal that combines the translational symmetry of a crystalline lattice with the point group symmetry. A space group is designated by a capital letter identifying the lattice type P A F etc. followed by the point group symbol in which the rotation and reflection elements are extended to include screw axes and glide planes. Note that the point group symmetry for a given space group can be determined by removing the cell centering symbol of the space group and replacing all screw axes by similar rotation axes and replacing all glide planes with mirror planes. The point group symmetry for a space group describes the true symmetry of its reciprocal lattice.

 Unit cell as used herein means the atoms in a crystal that are arranged in a regular repeating pattern in which the smallest repeating unit is called the unit cell. The entire structure can be reconstructed from knowledge of the unit cell which is characterized by three lengths a b and c and three angles and . The quantities a and b are the lengths of the sides of the base of the cell and is the angle between these two sides. The quantity c is the height of the unit cell. The angles and describe the angles between the base and the vertical sides of the unit cell.

 X ray diffraction pattern means the pattern obtained from X ray scattering of the periodic assembly of molecules or atoms in a crystal. X ray crystallography is a technique that exploits the fact that X rays are diffracted by crystals. X rays have the proper wavelength in the ngstr m range approximately 10cm to be scattered by the electron cloud of an atom of comparable size. Based on the diffraction pattern obtained from X ray scattering of the periodic assembly of molecules or atoms in the crystal the electron density can be reconstructed. Additional phase information can be extracted either from the diffraction data or from supplementing diffraction experiments to complete the reconstruction the phase problem in crystallography . A model is then progressively built into the experimental electron density refined against the data to produce an accurate molecular structure.

X ray structure coordinates define a unique configuration of points in space. Those of skill in the art understand that a set of structure coordinates for a protein or a protein ligand complex or a portion thereof define a relative set of points that in turn define a configuration in three dimensions. A similar or identical configuration can be defined by an entirely different set of coordinates provided the distances and angles between coordinates remain essentially the same. In addition a configuration of points can be defined by increasing or decreasing the distances between coordinates by a scalar factor while keeping the angles essentially the same.

 Crystal structure generally refers to the three dimensional or lattice spacing arrangement of repeating atomic or molecular units in a crystalline material. The crystal structure of a crystalline material can be determined by X ray crystallographic methods see for example Principles of Protein X Ray Crystallography by Jan Drenth Springer Advanced Texts in Chemistry Springer Verlag 2nd ed. February 1999 ISBN 0387985875 and Introduction to Macromolecular Crystallography by Alexander McPherson Wiley Liss Oct. 18 2002 ISBN 0471251224.

The present disclosure thus includes a crystalline form and a crystal structure of hepatocyte growth factor beta chain HGF complexed with Met receptor HGF Met and methods of using the HGF Met crystal structure and structural coordinates to identify homologous proteins and to design or identify agents that can modulate the function of HGF Met and or HGF Met complex. In some embodiments the crystalline form of HGF complexed with Met receptor diffracts X rays for a determination of atomic coordinates to a resolution of 5 or better. The present disclosure also includes the three dimensional configuration of points derived from the structure coordinates of at least a portion of an extracellular fragment of a Met receptor molecule or molecular complex as well as structurally equivalent configurations as described below. The three dimensional configuration includes points derived from structure coordinates representing the locations of a plurality of the amino acids defining the Met binding site for HGF the blades of the propeller and the PSI domain.

In some embodiments the three dimensional configuration includes points derived from structure coordinates representing the locations of the backbone atoms of a plurality of amino acids defining the Met or HGF Met complex binding site. Alternatively the three dimensional configuration includes points derived from structure coordinates representing the locations of the side chain and the backbone atoms other than hydrogens of a plurality of the amino acids defining the Met or HGF Met complex binding site including the Met binding site for HGF and the HGF binding site for Met.

The disclosure also includes the three dimensional configuration of points identifying other structural features of an extracellular fragment of the Met receptor. Those other structural features include the blades of the propeller structure and PSI domain. A plurality of amino acid residues have been identified as contributing to these structural features of Met receptor. In some embodiments the amino acid residues comprise those identified as corresponding to structural features as shown in .

Likewise the disclosure also includes the scalable three dimensional configuration of points derived from structure coordinates of molecules or molecular complexes that are structurally homologous to HGF Met complex or extracellular fragment of the Met receptor including the Sema domain as well as structurally equivalent configurations. Structurally homologous molecules or molecular complexes are defined below. Advantageously structurally homologous molecules can be identified using the structure coordinates of the HGF Met complex or extracellular fragment of the Met receptor according to a method of the disclosure.

The configurations of points in space derived from structure coordinates according to the disclosure can be visualized as for example a holographic image a stereodiagram a model or a computer displayed image and the disclosure thus includes such images diagrams or models.

The crystal structure and structural coordinates can be used in methods for example for obtaining structural information of a related molecule and for identifying and designing agents that modulate Met or HGF Met complex activity.

The coordinates of the present disclosure have been deposited in the RCSB Protein Data Bank under accession code PDBISHY.

The present disclosure includes a description of HGF and an extracellular fragment of Met including the Sema domain of the Met receptor.

The present disclosure includes a description of hepatocyte growth factor and or portions thereof. Hepatocyte growth factor comprises a 69 kDa alpha chain and 34 kDa beta chain. HGF is secreted as a single chain precursor form scHGF . The 69 kDa alpha chain comprise a N terminal finger domain and four kringle domains K1 K4 . A representative amino acid sequence of human HGF chain is shown in Table 4 SEQ ID NO 1 . The sequence of Table 4 has one amino acid change from wild type shown in Table 8 the cysteine at amino acid position 604 is changed to a serine. It would be expected that a wild type HGF would have an equivalent crystal structure. The amino acid numbering of the HGF chain is based on the numbering of the single chain precursor form. Numbers in brackets or preceded by a lower case c represent a numbering system based on reference to chymotrypsinogen.

The Met receptor is a tyrosine kinase and is part of a larger family of growth factor receptors with domain architecture similar to the Ron and Sea receptors. The extracellular portion of the Met receptor comprises N terminal 500 amino acids that fold into a Sema domain. A PSI domain follows the Sema domain and comprises about 50 amino acids and has 4 disulfide bonds. The PSI domain is connected to the transmembrane domain and extracellular kinase domain by 4 IPT domains. IPT domains are immunoglobulin like domains and are located C terminal to the PSI domain. Met becomes activated upon binding of a ligand such as HGF the receptor is phosphorylated and cells expressing activated Met are stimulated to migrate proliferate and or differentiate. Crosslinking of Met receptors to form dimers also activates the Met receptor. A representative example of an amino acid sequence of a Met receptor is shown in Table 5 and comprises an amino acid sequence of SEQ ID NO 2. A representative example of a sequence of an extracellular portion of the Met receptor including the Sema domain and the PSI domain is shown in Table 6 and comprises an amino acid sequence of SEQ ID NO 3. The extracellular fragment of the Met receptor having a sequence of SEQ ID NO 3 differs from wild type in that amino acids at positions 304 to 308 have been substituted to include a thrombin cleavage site. Another embodiment of an extracellular fragment of Met is provided in and comprises an amino acid sequence of SEQ ID NO 10. The numbering system of Met receptor is that of the Swiss Prot database as shown in Table 6.

Native or wild type HGF HGF HGF or Met receptor polypeptides are those polypeptides that have a sequence of a polypeptide obtained from nature. Native or wild type polypeptides include naturally occurring variants secreted or truncated forms. Some domains of HGF and or the Met receptor are known. Several isoforms of HGF are known such as isoform 1 isoform 2 isoform 3 isoform 4 and isoform 5. Representative sequences can be found at GenBank Accession Numbers NM000601 NM001010931 NM001010932 NM001010933 NM001010934 and NP000592.

The present disclosure also includes a polypeptide comprising consisting essentially of or consisting of a portion or fragment of the Met receptor. The polypeptide fragment includes amino acid residues from any of amino acid 1 to 25 residues to amino acid position 567 or residues corresponding to those positions. In some embodiments the polypeptide portion has the ability to bind to ligand HGF. The polypeptide portion may also be fused to heterologous polypeptide such as a peptide tag. Preferably the fusion polypeptide retains the ability to bind a ligand such as HGF.

The disclosure also provides a polypeptide comprising consisting essentially of or consisting of a portion or fragment of the Met receptor starting at amino acid residue 124 and ending at any one of amino acid residues 230 to 286 or residues corresponding to these residues. This polypeptide includes the amino acid residues that contact the HGF ligand and preferably the polypeptide has the ability to bind to a ligand such as HGF . The polypeptide portion or fragment may be fused to a heterologous polypeptide. Preferably the fusion protein can bind to a ligand such as HGF .

The present disclosure also includes variants of the Met receptor. Variants include those polypeptides that have amino acid substitutions deletions and additions. Amino acid substitutions can be made for example to replace cysteines and eliminate formation of disulfide bonds. Amino acid substitutions can also be made to change proteolytic cleavage sites. The variants also include those polypeptides that have at least one conservative amino acid substitution. In some embodiments a variant only has conservative amino acid substitutions. In some embodiments the Met receptor variant has about 1 25 conservative amino acid substitutions more preferably about 1 20 conservative amino acids substitutions more preferably about 1 10 conservative amino acid substitutions more preferably about 1 5 conservative amino acid substitutions and more preferably about 1 2 conservative amino acid substitutions. In some embodiments a Met receptor variant has at least 90 sequence identity to an extracellular domain fragment of the Met receptor such as SEQ ID NO 3 and has changes at amino acids other than those associated with the binding site for HGF on Met preferably the amino acid changes are only conservative substitutions. Other variants can be made at the Met binding site for HGF . In other embodiments the variants of the Met receptor bind HGF bind with the same or higher affinity than the wild type Met receptor.

A Met receptor variant or structural homolog or portions thereof may be fused to a heterologous polypeptide or compound. The heterologous polypeptide is a polypeptide that has a different function than that of the Met receptor. Examples of heterologous polypeptide include polypeptides that may act as carriers may extend half life may act as epitope tags may provide ways to detect or purify the fusion protein. Heterologous polypeptides include KLH albumin salvage receptor binding epitopes immunoglobulin constant regions and peptide tags. Peptide tags useful for detection or purification include FLAG gD protein polyhistidine tags hemaglutinin from influenza virus T7 tag S tag Strep tag chloramiphenicol acetyl transferase biotin glutathione S transferase green fluorescent protein and maltose binding protein. Compounds that can be combined with the Met receptor variants or structural homolog or portions thereof include radioactive labels protecting groups and carbohydrate or lipid moieties.

Variants of a Met receptor or extracellular fragment thereof can be prepared by introducing appropriate nucleotide changes into DNA encoding Met or the extracellular fragment or by synthesis of the desired polypeptide variants. HGF chain variants can be prepared by introducing appropriate nucleotide changes into DNA encoding HGF or by synthesis of the desired polypeptide variants. Variants can be made using standard methods.

Amino acid substitutions include one or more conservative amino acid substitutions. The term conservative amino acid substitution as used herein refers to an amino acid substitution which substitutes a functionally equivalent amino acid. Conservative amino acid changes result in silent changes in the amino acid sequence of the resulting polypeptide. For example one or more amino acids of a similar polarity act as functional equivalents and result in a silent alteration within the amino acid sequence of the peptide. In general substitutions within a group can be considered conservative with respect to structure and function. However the skilled artisan will recognize that the role of a particular residue is determined by its context within the three dimensional structure of the molecule in which it occurs. For example Cys residues may occur in the oxidized disulfide form which is less polar than the reduced thiol form. The long aliphatic portion of the Arg side chain can constitute a feature of its structural or functional role and this may be best conserved by substitution of a nonpolar rather than another basic residue. Also it will be recognized that side chains containing aromatic groups Trp Tyr and Phe can participate in ionic aromatic or cation pi interactions. In these cases substitution of one of these side chains with a member of the acidic or uncharged polar group may be conservative with respect to structure and function. Residues such as Pro Gly and Cys disulfide form can have direct effects on the main chain conformation and often may not be substituted without structural distortions.

Amino acid substitutions can be the result of replacing one amino acid with another amino acid having similar structural and or chemical properties such as the replacement of a leucine with a serine i.e. conservative amino acid replacements. Examples of conservative substitutions are shown in Table 9. The variation allowed can be determined by systematically making insertions deletions or substitutions of amino acids in the sequence and testing the resulting variants for activity exhibited by the native sequence.

Polynucleotide sequences encoding the polypeptides described herein can be obtained using standard recombinant techniques. Desired polynucleotide sequences may be isolated and sequenced from appropriate source cells. Alternatively polynucleotides can be synthesized using nucleotide synthesizer or PCR techniques. Once obtained sequences encoding the polypeptides or variant polypeptides are inserted into a recombinant vector capable of replicating and expressing heterologous polynucleotides in a host cell. Many vectors that are available and known in the art can be used for the purpose of the present invention. Selection of an appropriate vector will depend mainly on the size of the nucleic acids to be inserted into the vector and the particular host cell to be transformed with the vector. Each vector contains various components depending on its function amplification or expression of heterologous polynucleotide or both and its compatibility with the particular host cell in which it resides. The vector components generally include but are not limited to an origin of replication in particular when the vector is inserted into a prokaryotic cell a selection marker gene a promoter a ribosome binding site RBS a signal sequence the heterologous nucleic acid insert and a transcription termination sequence.

In general plasmid vectors containing replicon and control sequences which are derived from a species compatible with the host cell are used in connection with these hosts. The vector ordinarily carries a replication site as well as marking sequences which are capable of providing phenotypic selection in transformed cells. For example is typically transformed using pBR322 a plasmid derived from an species. pBR322 contains genes encoding ampicillin Amp and tetracycline Tet resistance and thus provides easy means for identifying transformed cells. pBR322 its derivatives or other microbial plasmids or bacteriophage may also contain or be modified to contain promoters which can be used by the microbial organism for expression of endogenous proteins.

In addition phage vectors containing replicon and control sequences that are compatible with the host microorganism can be used as transforming vectors in connection with these hosts. For example bacteriophage such as GEM 11 may be utilized in making a recombinant vector which can be used to transform susceptible host cells such as LE392.

Either constitutive or inducible promoters can be used in the present invention in accordance with the needs of a particular situation which can be ascertained by one skilled in the art. A large number of promoters recognized by a variety of potential host cells are well known. The selected promoter can be operably linked to cistron DNA encoding a polypeptide described herein by removing the promoter from the source DNA via restriction enzyme digestion and inserting the isolated promoter sequence into the vector of choice. Both the native promoter sequence and many heterologous promoters may be used to direct amplification and or expression of the target genes. However heterologous promoters are preferred as they generally permit greater transcription and higher yields of expressed target gene as compared to the native target polypeptide promoter.

Promoters suitable for use with prokaryotic hosts include the PhoA promoter the galactamase and lactose promoter systems a tryptophan trp promoter system and hybrid promoters such as the tac or the trc promoter. However other promoters that are functional in bacteria such as other known bacterial or phage promoters are suitable as well. Their nucleotide sequences have been published thereby enabling a skilled worker operably to ligate them to cistrons encoding the polypeptides or variant polypeptides Siebenlist et al. 1980 Cell 20 269 using linkers or adaptors to supply any required restriction sites.

In embodiments each cistron within a recombinant vector comprises a secretion signal sequence component that directs translocation of the expressed polypeptides across a membrane. In general the signal sequence may be a component of the vector or it may be a part of the polypeptide encoding DNA that is inserted into the vector. The signal sequence selected for the purpose of this invention should be one that is recognized and processed i.e. cleaved by a signal peptidase by the host cell. For prokaryotic host cells that do not recognize and process the signal sequences native to the heterologous polypeptides the signal sequence is substituted by a prokaryotic signal sequence selected for example from the group consisting of the alkaline phosphatase penicillinase Ipp or heat stable enterotoxin II STII leaders LamB PhoE PelB OmpA and MBP.

Prokaryotic host cells suitable for expressing polypeptides include Archaebacteria and Eubacteria such as Gram negative or Gram positive organisms. Examples of useful bacteria include e.g. Bacilli e.g. Enterobacteria species e.g. or . Preferably gram negative cells are used. Preferably the host cell should secrete minimal amounts of proteolytic enzymes and additional protease inhibitors may desirably be incorporated in the cell culture.

Besides prokaryotic host cells eukaryotic host cell systems are also well established in the art. Examples of invertebrate cells include insect cells such as S2 and Sf9 as well as plants and plant cells. Examples of useful mammalian host cell lines include Chinese hamster ovary CHO and COS cells. More specific examples include monkey kidney CV1 line transformed by SV40 COS 7 ATCC CRL 1651 Chinese hamster ovary cells DHFR CHO Urlaub and Chasin 77 4216 1980 mouse sertoli cells TM4 Mather 23 243 251 1980 and mouse mammary tumor MMT 060562 ATCC CCL51 .

Host cells are transformed or transfected with the above described expression vectors and cultured in conventional nutrient media modified as appropriate for inducing promoters selecting transformants or amplifying the genes encoding the desired sequences.

Transfection refers to the taking up of an expression vector by a host cell whether or not any coding sequences are in fact expressed. Numerous methods of transfection are known to the ordinarily skilled artisan for example CaPOprecipitation and electroporation. Successful transfection is generally recognized when any indication of the operation of this vector occurs within the host cell.

Transformation means introducing DNA into the prokaryotic host so that the DNA is replicable either as an extrachromosomal element or by chromosomal integrant. Depending on the host cell used transformation is done using standard techniques appropriate to such cells. The calcium treatment employing calcium chloride is generally used for bacterial cells that contain substantial cell wall barriers. Another method for transformation employs polyethylene glycol DMSO. Yet another technique used is electroporation.

Prokaryotic cells used to produce the polypeptides of the invention are grown in media known in the art and suitable for culture of the selected host cells. Examples of suitable media include luria broth LB plus necessary nutrient supplements. In preferred embodiments the media also contains a selection agent chosen based on the construction of the expression vector to selectively permit growth of prokaryotic cells containing the expression vector. For example ampicillin is added to media for growth of cells expressing ampicillin resistant gene.

Any necessary supplements besides carbon nitrogen and inorganic phosphate sources may also be included at appropriate concentrations introduced alone or as a mixture with another supplement or medium such as a complex nitrogen source. Optionally the culture medium may contain one or more reducing agents selected from the group consisting of glutathione cysteine cystamine thioglycollate dithioerythritol and dithiothreitol.

The prokaryotic host cells are cultured at suitable temperatures. For growth for example the preferred temperature ranges from about 20 C. to about 39 C. more preferably from about 25 C. to about 37 C. even more preferably at about 30 C. The pH of the medium may be any pH ranging from about 5 to about 9 depending mainly on the host organism. For the pH is preferably from about 6.8 to about 7.4 and more preferably about 7.0.

If an inducible promoter is used in the expression vector protein expression is induced under conditions suitable for the activation of the promoter. For example if a PhoA promoter is used for controlling transcription the transformed host cells may be cultured in a phosphate limiting medium for induction. A variety of other inducers may be used according to the vector construct employed as is known in the art.

Eukaryotic host cells are cultured under conditions suitable for expression of the HGF and or Met receptor polypeptides. The host cells used to produce the polypeptides may be cultured in a variety of media. Commercially available media such as Ham s F10 Sigma Minimal Essential Medium MEM Sigma RPMI 1640 Sigma and Dulbecco s Modified Eagle s Medium DMEM Sigma are suitable for culturing the host cells. In addition any of the media described in one or more of Ham et al. 197958 44 Barnes et al. 1980102 255 U.S. Pat. No. 4 767 704 4 657 866 4 927 762 4 560 655 or 5 122 469 WO 90 103430 WO 87 00195 and U.S. Pat. No. Re. 30 985 may be used as culture media for the host cells. Any of these media may be supplemented as necessary with hormones and or other growth factors such as insulin transferrin or epidermal growth factor salts such as sodium chloride calcium magnesium and phosphate buffers such as HEPES nucleotides such as adenosine and thymidine antibiotics such as GENTAMYCIN trace elements defined as inorganic compounds usually present at final concentrations in the micromolar range and glucose or an equivalent energy source. Other supplements may also be included at appropriate concentrations that would be known to those skilled in the art. The culture conditions such as temperature pH and the like are those previously used with the host cell selected for expression and will be apparent to the ordinarily skilled artisan.

Polypeptides described herein expressed in a host cell may be secreted into and or recovered from the periplasm of the host cells. Protein recovery typically involves disrupting the microorganism generally by such means as osmotic shock sonication or lysis. Once cells are disrupted cell debris or whole cells may be removed by centrifugation or filtration. The proteins may be further purified for example by affinity resin chromatography. Alternatively proteins can be transported into the culture media and isolated there from. Cells may be removed from the culture and the culture supernatant being filtered and concentrated for further purification of the proteins produced. The expressed polypeptides can be further isolated and identified using commonly known methods such as fractionation on immunoaffinity or ion exchange columns ethanol precipitation reverse phase HPLC chromatography on silica or on a cation exchange resin such as DEAE chromatofocusing SDS PAGE ammonium sulfate precipitation gel filtration using for example Sephadex G 75 hydrophobic affinity resins ligand affinity using a suitable antigen immobilized on a matrix and Western blot assay.

Polypeptides that are produced may be purified to obtain preparations that are substantially homogeneous for further assays and uses. Standard protein purification methods known in the art can be employed. The following procedures are exemplary of suitable purification procedures fractionation on immunoaffinity or ion exchange columns ethanol precipitation reverse phase HPLC chromatography on silica or on a cation exchange resin such as DEAE chromatofocusing SDS PAGE ammonium sulfate precipitation and gel filtration using for example Sephadex G 75.

The present disclosure provides crystals of and a crystal structure of HGF chain cocrystallized with a fragment of the Met receptor. In some embodiments the Met receptor fragment includes the Sema and PSI domain. The crystals were formed by contacting a mixture of purified HGF chain and the Met receptor extracellular fragment with a precipitant in a buffer. In some embodiments the crystal of a HGF and a Met receptor can be diffracted by X rays to determine atomic coordinates to a resolution of 5 or better. In some embodiments the precipitant was 12 15 polyethylene glycol 10 000. In some embodiments the crystals are formed from a HGF comprising SEQ ID NO 1 and a Met receptor fragment comprising SEQ ID NO 3.

The resulting crystals diffracted to 3.2 resolution Table 1 and have one 1 full complex containing one Met fragment and a single HGF chain in the asymmetric unit. The structure was refined to an R value of 20.9 Rfree 27.0 with good geometry. Of the 628 non glycine and non proline residues 97.6 have their main chain torsion angles in the most favored or the additionally allowed regions of the Ramachandran plot Laskowski et al. 1993 . The refined model includes residues 495 to 722 of HGF and residues 40 to 301 311 to 377 382 to 400 and 414 to 564 of Met. Although a number of glycosylation sites displayed electron density none of the sugars were modeled into the structure.

In a specific embodiment the structure of HGF complexed with a Met receptor extracellular fragment HGF Met was solved by molecular replacement with the program AMORE NAVAZC 1994 using the crystal structure of HGF chain alone as search model coordinates for HGF can be found in the RCSB Protein Data Bank under accession code PDB1UX3 . The crystals belonged to space group P2P2P with cell parameters of a 137.1 b 186.4 and c 66.7 and contained 1 complex of Met HGF chain in the asymmetric unit.

Each of the constituent amino acids in HGF Met is defined by a set of structural coordinates as set forth in Table 2. The coordinates and structure factors of the present disclosure have been deposited by the RCSB Protein Data Bank under Accession Code PDB 1SHY.

The term structure coordinates refers to Cartesian coordinates derived from mathematical equations related to the patterns obtained on diffraction of a monochromatic beam of X rays by the atoms scattering centers of a Met receptor or Met HGF in crystal form. The diffraction data are used to calculate an electron density map of the repeating unit of the crystal. The electron density maps are then used to establish the positions of the individual atoms of the Met receptor or protein ligand complex.

Slight variations in structure coordinates can be generated by mathematically manipulating the Met receptor or Met HGF complex structure coordinates. For example the structure coordinates as set forth in Table 2 could be manipulated by crystallographic permutations of the structure coordinates fractionalization of the structure coordinates integer additions or subtractions to sets of the structure coordinates inversion of the structure coordinates or any combination of the above. Alternatively modifications in the crystal structure due to mutations additions substitutions deletions and combinations thereof of amino acids or other changes in any of the components that make up the crystal could also yield variations in structure coordinates. Such slight variations in the individual coordinates will have little effect on overall shape. If such variations are within an acceptable standard error as compared to the original coordinates the resulting three dimensional shape is considered to be structurally equivalent. Structural equivalence is described in more detail below.

It should be noted that slight variations in individual structure coordinates of the HGF Met would not be expected to significantly alter the nature of chemical entities such as ligands that could associate with a binding site or other structural features of HGF or Met. In this context the phrase associating with refers to a condition of proximity between a ligand or portions thereof and a HGF or Met molecule or portions thereof. The association may be non covalent wherein the juxtaposition is energetically favored by hydrogen bonding van der Waals forces and or electrostatic interactions or it may be covalent.

The chain of HGF shares close to 40 sequence identity with the protease domain of plasmin a trypsin like serine protease. Structurally these enzymes can be described as globular proteins composed of two antiparallel barrel domains . Both of the barrels share the same general topology and are formed by six antiparallel strands with the N terminal four strands folding into a Greek key motif followed by two strands that form a hairpin. HGF like serine proteases is expressed as a zymogen like precursor. For serine proteases structural studies have shown that insertion of the N terminus resulting from the maturation process leads to allosteric rearrangements within the binding site for the protease substrate Perona and Craig 1995 Hedstrom 2002 . In serine proteases the newly formed N terminus forms a salt bridge with a nearby aspartate which leads to the formation of the oxyanion hole via the backbone NH s of two neighboring residues and the creation of a part of the S1 pocket . The activated form of the HGF chain as seen in the structure presented here is similar to the mature form of serine proteases superposition with plasmin pdb code 1BML yields an rmsd of about 1.3 for 212 C pairs. Superposition between HGF chain and plasminogen pdb code 1QRZ reveal differences in their activation domain Freer et al. 1970 and only 198 C pairs can be aligned well.

As with serine proteases the N terminal amine of HGF Val495 is inserted into the core of the C terminal barrel and forms a salt bridge with the carboxyl group of Asp672 which corresponds to Asp c194 in plasmin presumably leading to similar rearrangements which have been characterized in serine proteases. The region in HGF corresponding to this so called activation domain i.e. the loops that undergo conformational changes in serine proteases Freer et al. 1970 Huber and Bode 1978 form part of the binding surface with Met.

After maturation the and chains of HGF remain connected via a disulfide bond. Based on alignments of the kringle domains K1 to K4 of HGF chain and macrophage stimulating protein MSP the two cysteine residues responsible for the formation of this disulfide bond were identified as Cys487 on the chain of HGF and Cys604 on the chain Donate et al. 1994 . The crystal structure however raises the possibility of an alternative cysteine residue as the anchor for the HGF chain. The HGF chain has two 2 cysteine residues that do not have partners for the formation of disulfide bridges within the chain. Neither is conserved in plasmin or MSP and both are located on the periphery of the protein and are at least partially solvent exposed either could be potential partners for the formation of a disulfide bond with the chain while the other remains unpaired. The superposition of the plasmin structure that contains part of the chain and the HGF chain structure presented here shows that the distance between the C terminus of the plasmin chain and Cys604 of the HGF chain is 27 and thus only slightly shorter than the respective distance between the plasmin chain C terminus and Cys561 33 . In addition the sequence alignment between plasmin MSP and HGF shows that the chain cysteine forming the disulfide bond in plasmin and MSP is 13 and 15 residues away from the cleavage site in those proteins respectively but there are only 7 residues in HGF to span the distance from Cys487 to the cleavage site. Therefore due to distance requirements the C terminus of the HGF chain cannot follow the same path on the surface of the chain as it does in plasmin or MSP. Regardless of the position of the disulfide bond between the alpha and beta chain the overall structure or the structural model of the Met receptor or Met HGF complex is not impacted.

The analysis of crystal structure of the Sema domain of Met shows that it forms 7 bladed propeller with a diameter of a little more than 50 . The overall shape of the domain resembles a funnel with an inner diameter of about 25 between main chain atoms at the wide portion and 10 in the narrowest part . Generally in propellers each of the blades is formed by 4 antiparallel strands with strand A in the center of the propeller followed by strands B and C and with strand D forming the outermost strand of the blade. The blades are arranged in a circular fashion with the N terminal strand forming strand D of the last blade thus closing the propeller and stabilizing the overall structure . The AB and CD loops of each blade of the Met Sema domain form the relatively flat bottom face and the generally longer BC and DA loops form the top face of the propeller . In Met the position of the 6th and the 7th blade are off center with blade 7 being closer to the center of the barrel and blade 6 more distant. This gives the domain an overall oval shape.

The propeller of the Met Sema domain is structurally most closely related to the recently reported crystal structures of Sema4D Love et al. 2003 and Sema3A Antipenkov et al. 2003 . With the exception of the D strands in blades 3 and 5 the core of Sema4D and the Met Sema domain align well and the superposition of residues 40 to 519 with the Sema domain of Sema4D results in an rmsd of 1.6 for 303 atom pairs . The residues that form strand D5 in Sema4A are hydrogen bonded to strand D4 in Met thus the 4th blade of Met contains an extra strand while blade 5 is missing its strand D. While the core of both proteins align well the loops contain a number of insertions or deletions and generally adopt very different conformations. For example the Sema domains of Met and of Sema4D both have an insert of about 20 residues following strand D1. In Met these residues form a short antiparallel two stranded sheet while Sema4D has an helix . Of the 24 loops that connect the various strands only 3 have no insertions or deletions when comparing Met with Sema4D or Sema3A . Interestingly both the Semaphorins and the Met receptor have their largest insertion between strands C5 and A6 however these inserts are not related in structure or sequence to each other. In Sema3A this insert is involved in dimer formation and has been shown to be important for neuropilin binding Antipenko et al. 2003 . In the Met Sema domain some of the residues within the insert are disordered and not included in our model see . The function of the insert in Met is not known but its proximity to the HGF chain binding site discussed below suggests that it might be involved in binding to the chain of HGF.

The loop containing the cleavage site between the and chain of Met connects strand D4 to A5 it is disordered in the electron density. After processing of native Met the chain which forms the N terminal 4 blades of the Sema domain and the chain remain connected via at least 2 disulfide bonds. One disulfide bond is formed between residues Cys298 on strand D4 and Cys363 on C5 and the second is formed between Cys282 D3 and Cys409. This last cysteine is positioned in a disordered region of the insert after blade 4. The electron density does not allow tracing of the main chain in this area but there is sufficient density in the area of the S position to suggest that the disulfide bond is at least partially formed. With the exception of the N terminal Cys26 which is disordered in the presently disclosed crystal structure and has no potential partner in the construct that was used for crystallization all cysteine residues are engaged in disulfide bonds. It is unclear if Cys26 remains unpaired in the context of full length Met receptor or if it forms a disulfide bond with another cysteine that resides in the IPT domains. Besides the two disulfide bonds that covalently connect the and chain of the Sema domain the domain is stabilized by 5 additional disulfide bridges formed between Cys95 Cys10 Cys98 Cys159 Cys133 Cys141 Cys172 Cys175 and Cys385 Cys397 .

The surface of the Met Sema domain appears to be mostly negatively charged especially in the center of the bottom face of the Sema domain . This surface which is formed by a 20 residue insertion after strand D in the first blade and includes the rather irregular area of strand D in the third blade shows clusters of acidic residues. Monomeric full length Met was shown to bind to heparin Gherardi et al. 2003 . In the presently disclosed structure there are no dominant positively charged patches within the Met Sema domain however two arginines are located in the vicinity of the disordered furin cleavage site. Together with the 6 positively charged residues that reside in this loop these arginines could constitute a potential heparin binding site.

The last strand of the Sema domain C7 is followed immediately by the PSI domain. This domain with dimensions of about 20 15 contains four 4 disulfide bridges and is not an integral part of the Sema domain but rather an independent structural module. The small core of the domain is formed by a helix and a short two stranded antiparallel sheet that are connected via a disulfide bond and sandwich the side chain of Trp540. Superposition of this domain with the PSI domain of Sema4D results in an rmsd of 1.6 for 41 C pairs. The relative orientation between the Sema and the PSI domains in the Met and the Sema4D structures however is different and requires a rotation of about 40 for superposition.

The HGF chain binds to the Sema domain of Met at the bottom face of the propeller and forms contacts with residues that protrude from blades 2 and 3 . This is unlike other propeller containing receptors such as the integrins which bind their ligands utilizing the top face of the propeller Xiong et al. 2002 . Other Sema domain containing proteins such as Sema3A and Sema4D are also postulated to bind ligands using the top face Antipenko et al. 2003 Love et al. 2003 .

The interface between HGF chain and Met buries a total of 1770 of solvent accessible surface and is dominated by polar or charge charge interactions. The volume of the cavity of the pocket that is blocked by Met on the surface of HGF is about 380 cubic angstroms as determined by placing a number of atoms at the entrance of the pocket to close the pocket and using the program GRASP available from Columbia University at honigts columbia edu to calculate the volume of the resulting cavity.

Of the 20 HGF residues that have more that 10 buried in the interface 6 are positively charged 3 carry negative charges and 3 are aromatic. The positively charged amino acids include K516 R533 R647 R695 K649 and R702. The negatively charged amino acids include E656 E670 and D578. The aromatic amino acids include Y513 Y673 and Y619. See Table 3 .

Even more dramatic of the 18 residues contributing more than 10 of buried surface to the interface on the Met receptor 6 carry negative charges 4 are positively charged and 3 are aromatic. The positively charged amino acid residues include H148 R191 R218 and K223. The negatively charged amino acids include D127 D128 E168 E221 and D224. The aromatic amino acids include Y125 Y126 and F192. The combined buried surface of the hydrophobic residues Ala Val Leu Ile Met and Phe amounts to a mere 145 while the charged residues Asp Glu Lys Arg and His are responsible for more than 1 000 of buried surface. The resolution of the presently disclosed structure does not permit a detailed discussion of hydrogen bonds however the position of the side chains suggest the presence of at least 7 charged interactions between Met and the HGF chain. In some embodiments variants of HGF beta or Met receptor may be designed to preserve and or strengthen these charge interactions. In other embodiments small molecules may be designed to mimic the charge charge interaction of either the HGF binding site for Met or the Met binding site for HGF . See Table 3 .

On HGF the residues analogous to those responsible for substrate binding and catalysis in the related serine proteases form most of the interactions with Met. Serine proteases bind linear peptides to their recognition site and form hydrogen bonds to main chain atoms to position a specific peptide bond for proteolytic cleavage. In contrast the HGF chain binds to a series of protruding polar side chains from Met that comprise amino acid residues from 3 separate loops . The first of these loops includes residues 124 128 of Met and connects strands A2 and B2 of Met. This loop contains two tyrosines that are in the core of the binding interface and pack against Arg695 of HGF . The second loop of Met that contacts HGF chain includes residues Asp190 Arg191 and Phe192 and connects A3 and B3. Asp190 of Met forms charged interactions with Arg533 of HGF. The side chain of Arg191 of Met packs against Val692 and Pro693 of HGF with its hydrophobic portion and in the presently disclosed model is in hydrogen bonding distance to the side chains of Glu656 and Asp578 with its guanidinium group. This Asp578 corresponds to aspartic acid c102 in the catalytic triad of the related serine proteases. Furthermore the backbone of this loop forms hydrophilic interactions with Gln534 the residue homologous to the histidine c57 of the protease catalytic triad.

The third segment contributing to the interface also includes strand D in the third blade of the propeller an area that deviates from the classical propeller fold . Strand D is interrupted and has a short helical insert within its strand. This unusual conformation presents a number of residues towards the surface of the HGF chain. The most prominent interaction formed by this loop involves Glu221 which extends its side chain towards what would be the S1 binding site in serine proteases . The carboxylate of Glu221 forms an extensive network of polar interactions with the side chain of Tyr673 corresponding to the serine c195 of the catalytic triad in proteases as well as the backbone amides of Gly694 and Gly696 however unlike in serine proteases where the S1 pocket is filled by the specificity determining P1 residue it does not penetrate deeply into the S1 pocket. As a result of the exchange of Ser to Tyr in position 673 c195 the entrance of the S1 pocket is much smaller than in the related proteases. Thus Glu221 projects towards the entrance of the S1 pocket from a different angle when compared to a complex between trypsin and bovine pancreatic trypsin inhibitor BPTI and does not fill the pocket . Therefore a large cavity presumably filled with a number of water molecules remains in HGF chain.

The structures of the HGF chain bound to Met and HGF in its unbound state as disclosed in application U.S. Ser. No. 60 569 301 filed May 6 2004 which is hereby incorporated by reference are very similar and superimpose with an rmsd of less than 0.8 for 218 C pairs. The only significant differences between the two structures involve residues that are close to the Met binding interface. The backbone of residues 645 to 651 is disordered in unliganded HGF but well defined in the structure of the complex where it adopts a conformation that is commonly observed in serine proteases . The side chains of the basic residues Arg647 and Lys649 are not well defined in the electron density but project towards the Met binding interface and in presently disclosed model participate in the formation of the complex. Arg695 also undergoes a change in conformation upon Met binding. This residue located in the center of the receptor binding interface packs against Tyr125 of Met and its backbone forms part of the S1 pocket. In the structure of unliganded HGF the conformation of the backbone in this area is different and the side chain of Arg695 projects toward the disordered region of the structure. It is conceivable that the binding event alters the position of Arg695 which in turn allows residues 645 to 651 to adopt a more stable conformation. The tunnel or void identified in the crystal structure of HGF is also seen in the cocrystal.

Various computational analyses can be used to determine whether a molecule or portions of the molecule defining structure features are structurally equivalent defined in terms of its three dimensional structure to all or part of a Met receptor or HGF Met complex or its ligand binding sites. Such analyses may be carried out in current software applications such as the Molecular Similarity application of QUANTA Molecular Simulations Inc. San Diego Calif. Version 4.1 and as described in the accompanying User s Guide.

The Molecular Similarity application permits comparisons between different structures different conformations of the same structure and different parts of the same structure. A procedure used in Molecular Similarity to compare structures comprises 1 loading the structures to be compared 2 defining the atom equivalences in these structures 3 performing a fitting operation and 4 analyzing the results.

One structure is identified as the target i.e. the fixed structure all remaining structures are working structures i.e. moving structures . Since atom equivalency within QUANTA is defined by user input for the purpose of this disclosure equivalent atoms are defined as protein backbone atoms N C C and O for all conserved residues between the two structures being compared. A conserved residue is defined as a residue that is structurally or functionally equivalent. Only rigid fitting operations are considered.

When a rigid fitting method is used the working structure is translated and rotated to obtain an optimum fit with the target structure. The fitting operation uses an algorithm that computes the optimum translation and rotation to be applied to the moving structure such that the root mean square difference of the fit over the specified pairs of equivalent atom is an absolute minimum. This number given in Angstroms is reported by QUANTA.

Structurally equivalent crystal structures have portions of the two molecules that are substantially identical within an acceptable margin of error. The margin of error can be calculated by methods known to those of skill in the art In some embodiments any molecule or molecular complex or any portion thereof that has a root mean square deviation of conserved residue backbone atoms N C C O of less than about 0.70 preferably 0.5 . For example structurally equivalent molecules or molecular complexes are those that are defined by the entire set of structure coordinates listed in Table 2 a root mean square deviation from the conserved backbone atoms of those amino acids of not more than 0.70 preferably 0.5 . The term root mean square deviation means the square root of the arithmetic mean of the squares of the deviations. It is a way to express the deviation or variation from a trend or object. For purposes of this disclosure the root mean square deviation defines the variation in the backbone of a protein from the backbone of HGF Met complex as defined by the structure coordinates of the complex as described herein or a defining structural feature thereof.

Structure coordinates can be used to aid in obtaining structural information about another crystallized molecule or molecular complex. The method of the disclosure allows determination of at least a portion of the three dimensional structure of molecules or molecular complexes that contain one or more structural features that are similar to structural features of at least a portion of Met receptor or HGF Met complex. These molecules are referred to herein as structurally homologous to Met receptor or HGF Met. Similar structural features can include for example regions of amino acid identity conserved active site or binding site motifs and similarly arranged secondary structural elements e.g. binding sites for HGF PSI domain IPT domain and propellor blades of the Sema domain .

Optionally structural homology is determined by aligning the residues of the two amino acid sequences to optimize the number of identical amino acids along the lengths of their sequences gaps in either or both sequences are permitted in making the alignment in order to optimize the number of identical amino acids although the amino acids in each sequence must nonetheless remain in their proper order. Two amino acid sequences are compared using the BLAST program version 2.0.9 of the BLAST 2 search algorithm as described by Tatusova et al. 56 and available at http www.ncbi.nlm.nih.gov BLAST . Preferably the default values for all BLAST 2 search parameters are used including matrix BLOSUM62 open gap penalty 11 extension gap penalty 1 gap x dropoff 50 expect 10 wordsize 3 and filter on. In the comparison of two amino acid sequences using the BLAST search algorithm structural similarity is referred to as identity. 

In some embodiments a structurally homologous molecule is a protein that has an amino acid sequence sharing at least 80 identity with a native or recombinant amino acid sequence of Met preferably an extracellular fragment of the Met receptor comprising a sequence of SEQ ID NO 3 or SEQ ID NO 10. An extracellular fragment of Met receptor comprising a sequence of SEQ ID NO 3 has amino acid substitutions at positions 304 308 to insert a thrombin cleavage site. In some embodiments a Met receptor has a sequence of SEQ ID NO 3 and the structurally homologous molecule is a variant that has a sequence identity to SEQ ID NO 3 of at least 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 or 99 or greater. In some embodiments the Met receptor variant or structurally homologous molecule has one or more conservative amino acid substitutions preferably only conservative amino acid substitutions and retains the structure of the binding site for HGF . In some embodiments the Met receptor variant has about 1 25 conservative amino acid substitutions more preferably about 1 20 conservative amino acids substitutions more preferably about 1 10 conservative amino acid substitutions more preferably about 1 5 conservative amino acid substitutions and more preferably about 1 2 conservative amino acid substitutions. Preferably the variant retains at least one or more domains such as the binding site for HGF . More preferably a protein that is structurally homologous to Met includes at least one contiguous stretch of at least 50 amino acids that shares at least 80 amino acid sequence identity with the analogous portion of the native or recombinant Met. Methods for generating structural information about the structurally homologous molecule or molecular complex are well known and include for example molecular replacement techniques.

Therefore in another embodiment this disclosure provides a method of utilizing molecular replacement to obtain structural information about a molecule or molecular complex whose structure is unknown comprising 

 a generating an X ray diffraction pattern from a crystallized molecule or molecular complex of unknown or incompletely known structure and or b applying at least a portion of the structural coordinates of the Met receptor or HGF Met complex to the X ray diffraction pattern to generate a three dimensional electron density map of the molecule or molecular complex whose structure is unknown or incompletely known.

By using molecular replacement all or part of the structure coordinates of the Met receptor and or HGF Met complex as provided by this disclosure can be used to determine the unsolved structure of a crystallized molecule or molecular complex more quickly and efficiently than attempting to determine such information ab initio. Coordinates of structural features of the Met receptor can be utilized including the Sema domain PSI domain and the binding site for HGF .

Molecular replacement can provide an accurate estimation of the phases for an unknown or incompletely known structure. Phases are one factor in equations that are used to solve crystal structures and this factor cannot be determined directly. Obtaining accurate values for the phases by methods other than molecular replacement can be a time consuming process that involves iterative cycles of approximations and refinements and greatly hinders the solution of crystal structures. However when the crystal structure of a protein containing at least a structurally homologous portion has been solved molecular replacement using the known structure provide a useful estimate of the phases for the unknown or incompletely known structure.

Thus this method involves generating a preliminary model of a molecule or molecular complex whose structure coordinates are unknown by orienting and positioning the relevant portion of the Met receptor and or HGF Met complex within the unit cell of the crystal of the unknown molecule or molecular complex. This orientation or positioning is conducted so as best to account for the observed X ray diffraction pattern of the crystal of the molecule or molecular complex whose structure is unknown. Phases can then be calculated from this model and combined with the observed X ray diffraction pattern amplitudes to generate an electron density map of the structure. This map in turn can be subjected to established and well known model building and structure refinement techniques to provide a final accurate structure of the unknown crystallized molecule or molecular complex see for example Lattman 1985115 55 77 .

Structural information about a portion of any crystallized molecule or molecular complex that is sufficiently structurally homologous to a portion of the Met receptor and or HGF Met can be resolved by this method. In addition to a molecule that shares one or more structural features with the Met receptor such as the Sema domain and or HGF Met as described above a molecule that has similar bioactivity such as the same catalytic activity substrate specificity or ligand binding activity as the Met receptor and or HGF Met may also be sufficiently structurally homologous to a portion of the Met receptor and or HGF Met to permit use of the structure coordinates of HGF Met to solve its crystal structure or identify structural features that are similar to those identified in the Met receptor described herein. It will be appreciated that amino acid residues in the structurally homologous molecule identified as corresponding to the Met receptor structural feature may have different amino acid numbering.

In one embodiment of the disclosure the method of molecular replacement is utilized to obtain structural information about a molecule or molecular complex wherein the molecule or molecular complex includes at least one HGF Met or Met receptor subunit or homolog. In the context of the present disclosure a structural homolog of the Met receptor or HGF Met is a protein that contains one or more amino acid substitutions deletions additions or rearrangements with respect to the amino acid sequence of HGF Met complex or Met receptor but that when folded into its native conformation exhibits or is reasonably expected to exhibit at least a portion of the tertiary three dimensional structure of at least a portion of the Met receptor and or HGF Met complex. A portion of the Met receptor includes the Sema domain PSI domain IPT domains and binding site for HGF and combinations thereof.

For example structurally homologous molecules can contain deletions or additions of one or more contiguous or noncontiguous amino acids such as a loop or a domain. Structurally homologous molecules also include modified extracellular fragment of the Met receptor and or HGF Met molecules that have been chemically or enzymatically derivatized at one or more constituent amino acid including side chain modifications backbone modifications and N and C terminal modifications including acetylation hydroxylation methylation amidation and the attachment of carbohydrate or lipid moieties cofactors and like modifications. It will be appreciated that amino acid residues in the structurally homologous molecule identified as corresponding to extracellular fragment of the Met receptor or other structural feature of the Sema domain of the Met receptor may have different amino acid numbering.

A heavy atom derivative of HGF Met is also included as a HGF Met homolog. The term heavy atom derivative refers to derivatives of HGF Met produced by chemically modifying a crystal of HGF or Met or both. In practice a crystal is soaked in a solution containing heavy metal atom salts or organometallic compounds e.g. lead chloride gold thiomalate thiomersal or uranyl acetate which can diffuse through the crystal and bind to the surface of the protein. The location s of the bound heavy metal atom s can be determined by X ray diffraction analysis of the soaked crystal. This information in turn is used to generate the phase information used to construct three dimensional structure of the protein Blundell et al. 1976 Academic Press San Diego Calif. .

The structure coordinates of HGF Met provided by this disclosure are particularly useful in solving the structure of Met variants. Variants may be prepared for example by expression of Met cDNA previously altered in its coding sequence by oligonucleotide directed mutagenesis as described herein. Variants may also be generated by site specific incorporation of unnatural amino acids into Met proteins using known biosynthetic methods Noren et al. 1989244 182 88 . In this method the codon encoding the amino acid of interest in wild type Met is replaced by a blank nonsense codon TAG using oligonucleotide directed mutagenesis. A suppressor tRNA directed against this codon is then chemically aminoacylated in vitro with the desired unnatural amino acid. The aminoacylated tRNA is then added to an in vitro translation system to yield a mutant Met with the site specific incorporated unnatural amino acid.

The structure coordinates of HGF Met are also particularly useful to solve or model the structure of crystals of HGF Met HGF variants Met variants or Met homologs or HGF homologs which are co complexed with a variety of ligands. HGF is an inhibitor of full length HGF and can be used to identify or design other like inhibitors. This approach enables the determination of the optimal sites for interaction between ligand entities including candidate HGF or Met ligands. Potential sites for modification within the various binding sites of the molecule can also be identified. This information provides an additional tool for determining more efficient binding interactions for example increased hydrophobic or polar interactions between Met and a ligand. For example high resolution X ray diffraction data collected from crystals exposed to different types of solvent allows the determination of where each type of solvent molecule resides. Small molecules that bind tightly to those sites can then be designed and synthesized and tested for their Met affinity such as inhibition activity.

In another embodiment homology modeling can be conducted using the structural coordinates of HGF and or the Met receptor and a program designed to generate models of structures such as Protein Explorer Swiss Model or RASMOL. The programs can provide a structural model of a homolog or variant of HGF and or Met by providing the structural coordinates such as provided in Table 2 and an alignment of the sequences.

All of the complexes referred to above may be studied using well known X ray diffraction techniques and may be refined versus 1.5 3.5 resolution X ray data to an R factor of about 0.30 or less using computer software such as X PLOR Yale University distributed by Molecular Simulations Inc. see for example Blundell et al. 1976 Academic Press San Diego Calif. and Vol. 114 115 H. W. Wyckoff et al. eds. Academic Press 1985 . This information may thus be used to optimize known Met modulators and more importantly to design new Met modulators.

The disclosure also includes the unique three dimensional configuration defined by a set of points defined by the structure coordinates for a molecule or molecular complex structurally homologous to an extracellular fragment of the Met receptor or HGF Met complex as determined using the method of the present disclosure structurally equivalent configurations and magnetic storage media including such set of structure coordinates.

Using homology modeling a computer model of a HGF Met complex or Met receptor homolog can be built or refined without crystallizing the homolog. First a preliminary model of the homolog is created by sequence alignment with HGF Met or an extracellular fragment of Met secondary structure prediction the screening of structural libraries or any combination of those techniques. Computational software may be used to carry out the sequence alignments and the secondary structure predictions. Programs available for such an analysis include Protein Explorer eg available at molvissdsc.edu.protexpl.frontdoor.htm Swiss Model eg available at swissmodel.expasy.org and RASMOL.Structural incoherences e.g. structural fragments around insertions and deletions can be modeled by screening a structural library for peptides of the desired length and with a suitable conformation. For prediction of the side chain conformation a side chain rotamer library may be employed. If the homolog has been crystallized the final homology model can be used to solve the crystal structure of the homolog by molecular replacement as described above. Next the preliminary model is subjected to energy minimization to yield an energy minimized model. The energy minimized model may contain regions where stereochemistry restraints are violated in which case such regions are remodeled to obtain a final homology model. The homology model is positioned according to the results of molecular replacement and subjected to further refinement including molecular dynamics calculations.

Potent and selective ligands that modulate activity antagonists and agonists are identified using the three dimensional model of the Met binding site for HGF and or other structural features produced using the coordinates of a cocrystal of HGF with Met or a fragment thereof such as provided in Table 2. Using this model ligands that interact with the Met binding site for HGF are identified and the result of the interactions is modeled. In some embodiments agents identified as candidate molecules for modulating the activity of HGF Met and or HGF Met can be screened against known bioassays. For example the ability of an agent to inhibit the anti apoptotic effects of Met can be measured using assays known in the art or for example the assays disclosed in the Examples. Using the modeling information and the assays described one can identify agents that possess HGF Met and or HGF Met modulating properties.

The methods of the disclosure also include methods of identifying molecules that mimic HGF binding to a ligand such as the Met receptor or Met receptor binding to HGF or both but do not activate the HGF Met signaling pathway. HGF is an inhibitor of full length HGF and can be used to identify or design other like inhibitors. These molecules can be identified using the three dimensional model of HGF Met complex using the coordinates of Tables 7 and 2.

In another embodiment a candidate modulator can be identified using a biological assay such as binding to HGF and or HGF modulating Met phosphorylation or modulating HGF induced cell migration. The candidate modulator can then serve as a model to design similar agents and or to modify the candidate modulator for example to improve characteristics such as binding to HGF or Met receptor. Design or modification of candidate modulators can be accomplished using the crystal structure coordinates and available software.

Applicants disclosure provides information inter alia about the shape and structure of the structural binding site of Met for HGF in the presence or absence of a modulator. Binding sites are of significant utility in fields such as drug discovery. The association of natural ligands or substrates with the binding sites of their corresponding receptors or enzymes is the basis of many biological mechanisms of action. Similarly many drugs exert their biological effects through association with the binding sites of receptors and enzymes. Such associations may occur with all or any part of the binding site. An understanding of such associations helps lead to the design of drugs having more favorable associations with their target and thus improved biological effects. Therefore this information is valuable in designing potential modulators of HGF Met and or HGF Met binding sites as discussed in more detail below.

The amino acid constituents of a Met or HGF Met binding site as defined herein are positioned in three dimensions. In one aspect the structure coordinates defining a binding site of Met or HGF Met include structure coordinates of all atoms in the constituent amino acids in another aspect the structure coordinates of a binding site include structure coordinates of just the backbone atoms of the constituent atoms.

In some embodiments the HGF structural binding site for Met for example includes the amino acids highlighted in as well as those identified in Table 3. In some embodiments the amino acid residues identified in the HGF binding site for Met comprise consist essentially of or consist of at least one or more or all amino acid residue corresponding to residues 513 516 533 534 537 578 619 647 649 656 669 670 673 692 693 694 695 696 697 699 or 702 or mixtures thereof. In some embodiments the amino acid residues identified in the HGF binding site for Met comprise consist essentially of or consist of at least one or more or all amino acid residue corresponding to residues Y513 K516 R533 Q 534 P537 D 578 Y619 R647 K 649 E656 C669 E670 Y673 V692 P693 G694 R695 G 696 C697 I699 or R702 or mixtures thereof or conservative amino acid substitutions thereof. The cocrystallization studies indicate that amino acids residues 645 to 651 are better ordered in the HGF Met complex and further indicate that K649 contacts the Met receptor. In other embodiments the amino acid residues identified in the HGF binding site for Met comprise consist essentially of or consist of at least one or more or all amino acid residue corresponding to residues Y513 K516 R533 Q534 F536 P537 S538 R539 D578 Y619 R647 K649 E656 P668 C669 E670 Y673 V692 P693 G694 R695 C696 C697 I699 R702 or mixtures thereof or conservative amino acid substitutions thereof. These residues may participate in polar or nonpolar interactions with charged residues on Met.

The term Met structural binding site includes all or a portion of a molecule or molecular complex whose shape is sufficiently similar to at least a portion of a binding site of Met for HGF as to be expected to bind HGF or related structural analogs. A structurally equivalent ligand binding site is defined by a root mean square deviation from the structure coordinates of the backbone atoms of the amino acids that make up binding sites in Met of at most about 0.70 preferably about 0.5 . In some embodiments the Met structural binding site for HGF for example includes the amino acids highlighted in as well as those identified in Table 3. In some embodiments a structural binding site for HGF on the Met receptor comprises consists essentially of or consists of at least one or more or all amino acid residues corresponding to residues 124 128 148 167 190 192 218 220 to 223 229 to 230 or 286 414 or mixtures thereof. In some embodiments a structural binding site for HGF on the Met receptor comprises consists essentially of or consists of at least one or more or all amino acid residue corresponding to residues T124 Y125 Y126 D127 D128 H148 E167 E168 D190 R191 F192 R218 K220 E221 T222 K223 D224 M227 L229 T230 I284 S286 D414 or mixtures thereof or conservative substitutions thereof. These residues may participate in polar interactions with charged residues on HGF .

The binding site of HGF or Met may be defined by those amino acids whose backbone atoms are situated within about 5 of one or more constituent atoms of a bound substrate or ligand. In yet another alternative the binding site for HGF on Met receptor can be defined by those amino acids whose backbone atoms are situated within a sphere centered on the coordinates representing the alpha carbon atom of amino acid E221 on the Met receptor the sphere having a radius of about 5 6 for example 5.8 .

Other structural features include the blades of the propellor of the Sema domain of the Met receptor and those that define the void in the center of the propellor as well as the domains of the Met receptor including PSI and IPT domains. Amino acid residues that form these structural features can be found in .

Computational techniques can be used to screen identify select design ligands and combinations thereof capable of associating with Met or structurally homologous molecules. Candidate modulators of HGF HGF and or Met may be identified using functional assays such as binding to HGF or Met and novel modulators designed based on the structure of the candidate molecules so identified. Knowledge of the structure coordinates for Met or HGF Met permits for example the design the identification of synthetic compounds and like processes and the design the identification of other molecules and like processes that have a shape complementary to the conformation of the Met and or HGF binding sites. In particular computational techniques can be used to identify or design ligands such as agonists and or antagonists that associate with a Met and or HGF binding site. Antagonists may bind to or interfere with all or a portion of an active site of Met and can be competitive non competitive or uncompetitive inhibitors. Once identified and screened for biological activity these agonists antagonists and combinations thereof may be used therapeutically and or prophylactically for example to block Met activity and thus prevent the onset and or further progression of diseases associated with Met activity. Structure activity data for analogues of ligands that bind to or interfere with Met binding sites and or HGF binding sites can also be obtained computationally.

In some embodiments an antagonist or agonist may be a molecule that mimics either HGF or the Met receptor such that it binds to either the Met receptor or HGF or both. HGF is an inhibitor of full length HGF and can be used to identify or design other like inhibitors. In some embodiments if the molecule is an antagonist it binds to either receptor or both but does not activate the Met receptor. As the structural information provided herein indicates that the interaction of the HGF with the Met receptor includes charge charge interactions in some embodiments agonists or antagonists can be designed to include components that preserve and or strengthen the charge interactions. Such antagonists or agonists would include components that are able to interact for example hydrogen bond with the charged amino acids found in either the HGF or Met receptor binding site or both. In some embodiments for HGF antagonist or agonist molecules are designed or selected that can interact with at least one or more or all amino acid residues that comprise consist essentially of or consist of amino acid residues corresponding to amino acid residues Y513 K516 R533 Y619 R647 R695 K649 R702 E656 E670 Y673 or D578 or mixtures thereof. In some embodiments for the Met receptor antagonist or agonist molecules are designed or selected that can interact with at least one or more or all amino acid residues that comprise consist essentially of or consist of amino acid residues corresponding to amino acid residues Y125 Y126 H148 R191 R218 K223 D127 D128 E168 F192 E221 or D224 or mixtures thereof. In other embodiments another criteria that may be utilized in the design of modulators is whether the modulator can fit into the binding site cavity on HGF that is blocked by Met. The volume of that cavity is about 380 cubic angstroms. The volume of the cavity can be determined by placing atoms in the entrance of the pocket close to the surface and using a program like GRASP to calculate the volume of those atoms.

Data stored in a machine readable storage medium that is capable of displaying a graphical three dimensional representation of the structure of Met or a structurally homologous molecule or molecular complex as identified herein or portions thereof may thus be advantageously used for drug discovery. The structure coordinates of the ligand are used to generate a three dimensional image that can be computationally fit to the three dimensional image of Met HGF Met complex or a structurally homologous molecule. The three dimensional molecular structure encoded by the data in the data storage medium can then be computationally evaluated for its ability to associate with ligands. When the molecular structures encoded by the data is displayed in a graphical three dimensional representation on a computer screen the protein structure can also be visually inspected for potential association with ligands.

One embodiment of the method of drug design involves evaluating the potential association of a candidate ligand with Met such as HGF or a structurally homologous molecule or homologous complex particularly with a binding site on the Met receptor. The method of drug design thus includes computationally evaluating the potential of a selected ligand to associate with any of the molecules or molecular complexes set forth above. This method includes the steps of a employing computational means for example such as a programmable computer including the appropriate software known in the art or as disclosed herein to perform a fitting operation between the selected ligand and a ligand binding site or a pocket nearby the ligand binding site of the molecule or molecular complex and b analyzing the results of the fitting operation to quantify the association between the ligand and the ligand binding site. Optionally the method further comprises analyzing the ability of the selected ligand to interact with charged amino acids in either the HGF and or Met receptor binding sites. Other structural features of the Met receptor and or HGF Met complex can also be analyzed in the same manner.

In another embodiment the method of drug design involves computer assisted design of ligand that associate with Met HGF Met its homologs or portions thereof. Ligands can be designed in a step wise fashion one fragment at a time or may be designed as a whole or de novo. Ligands can be designed based on the structure of molecules that can modulate at least one biological function of HGF or Met.

In some embodiments to be a viable drug candidate the ligand identified or designed according to the method must be capable of structurally associating with at least part of a Met binding site and or HGF binding site and must be able sterically and energetically to assume a conformation that allows it to associate with the Met binding site and or HGF binding site. Non covalent molecular interactions important in this association include hydrogen bonding van der Waals interactions hydrophobic interactions and or electrostatic interactions. In some embodiments agents may contact at least one amino acid position in the Met binding site for HGF . Conformational considerations include the overall three dimensional structure and orientation of the ligand in relation to the ligand binding site and the spacing between various functional groups of a ligand that directly interact with the Met binding site and or HGF binding site or homologs thereof.

Optionally the potential binding of a ligand to a Met binding site and or HGF binding site is analyzed using computer modeling techniques prior to the actual synthesis and testing of the ligand. If these computational experiments suggest insufficient interaction and association between it and the Met binding site and or HGF binding site testing of the ligand is obviated. However if computer modeling indicates a strong interaction the molecule may then be synthesized and tested for its ability to bind to or interfere with a Met binding site and or HGF binding site. Binding assays to determine if a compound actually modulates with Met activity can also be performed and are well known in the art.

Several methods can be used to screen ligands or fragments for the ability to associate with a Met binding site and or HGF binding site. This process may begin by visual inspection of for example a Met binding site and or HGF binding site on the computer screen based on the Met or HGF Met structure coordinates or other coordinates which define a similar shape generated from the machine readable storage medium. Selected ligands may then be positioned in a variety of orientations or docked within the binding site. Docking may be accomplished using software such as QUANTA and SYBYL followed by energy minimization and molecular dynamics with standard molecular mechanics force fields such as CHARMM and AMBER.

Specialized computer programs may also assist in the process of selecting ligands. Examples include GRID Hubbard S. 19996 711 4 MCSS Miranker et al. 199111 29 34 available from Molecular Simulations San Diego Calif. AUTODOCK Goodsell et al. 19908 195 202 available from Scripps Research Institute La Jolla Calif. and DOCK Kuntz et al. 1982161 269 88 available from University of California San Francisco Calif.

For example an examination of the structure of HGF shows a tunnel or void formed by amino acid residues 634 660 670 673 691 and 693 706. A molecule with an indole ring can fit within this void as determined by manual docking

Met binding ligands can be designed to fit a Met binding site and or HGF binding site optionally as defined by the binding of a known modulator or one identified as modulating the activity of HGF or Met. Examples of tyrosine kinase inhibitor compounds are disclosed in U.S. Pat. No. 6 696 463. There are many ligand design methods including without limitation LUDI Bohm 19926 61 78 available from Molecular Simulations Inc. San Diego Calif. LEGEND Nishibata Y. and Itai A. 199336 2921 8 available from Molecular Simulations Inc. San Diego Calif. LeapFrog available from Tripos Associates St. Louis Mo. and SPROUT Gillet et al. 19937 127 53 available from the University of Leeds UK.

Once a compound has been designed or selected by the above methods the efficiency with which that ligand may bind to or interfere with a Met binding site and or HGF binding site may be tested and optimized by computational evaluation. For example an effective Met binding site ligand and or HGF binding site should preferably demonstrate a relatively small difference in energy between its bound and free states i.e. a small deformation energy of binding . Thus an efficient Met binding site ligand should preferably be designed with a deformation energy of binding of not greater than about 10 to about 15 kcal mole such as about 12 kcal mole preferably not greater than about 8 to about 12 kcal mole such as about 10 kcal mole and more preferably not greater than about 5 to about 10 kcal mole such as about 7 kcal mole. Met binding site and or HGF binding site ligands may interact with the binding site in more than one conformation that is similar in overall binding energy. In those cases the deformation energy of binding is taken to be the difference between the free energy of the ligand and the average energy of the conformations observed when the ligand binds to the protein.

A ligand designed or selected as binding to or interfering with a Met binding site and or HGF binding site may be further computationally optimized so that in its bound state it would preferably lack repulsive electrostatic interaction with the target enzyme and with the surrounding water molecules. Such non complementary electrostatic interactions include repulsive charge charge dipole dipole and charge dipole interactions.

Specific computer software is available to evaluate compound deformation energy and electrostatic interactions. Examples of programs designed for such uses include Gaussian 94 revision C M. J. Frisch Gaussian Inc. Pittsburgh Pa. AMBER version 4.1 P. A. Kollman University of California at San Francisco QUANTA CHARMM Molecular Simulations Inc. San Diego Calif. Insight II Discover Molecular Simulations Inc. San Diego Calif. DelPhi Molecular Simulations Inc. San Diego Calif. and AMSOL Quantum Chemistry Program Exchange Indiana University . These programs can be implemented for instance using a Silicon Graphics workstation such as an Indigo2 with IMPACT graphics. Other hardware systems and software packages will be known to those skilled in the art.

Another approach encompassed by this disclosure is the computational screening of small molecule databases for ligands or compounds that can bind in whole or in part to a Met binding site and or HGF binding site. In this screening the quality of fit of such ligands to the binding site may be judged either by shape complementarity or by estimated interaction energy Meng et al. 199213 505 24 . In addition these small molecule databases can be screened for the ability to interact with the charged amino acids in the Met binding site and or HGF binding site as identified herein.

A compound that is identified or designed as a result of any of these methods can be obtained or synthesized and tested for its biological activity for example binding and or inhibition of HGF and or Met activity.

Another method involves assessing agents that are antagonists or agonists of the Met receptor. A method comprises applying at least a portion of the crystallography coordinates of a cocrystal of HGF and Met such as provided in Tables 7 and or 2 to a computer algorithm that generates a three dimensional model of HGF Met complex or the Met receptor suitable for designing molecules that are antagonists or agonists and searching a molecular structure database to identify potential antagonists or agonists. In some embodiments a portion of the structural coordinates of a cocrystal of HGF and Met such as provided in Tables 7 and or 2 that define a structural feature for example binding site for HGF may be utilized. The method may further comprise synthesizing or obtaining the agonist or antagonist and contacting the agonist or antagonist with the Met receptor and or HGF and selecting the antagonist or agonist that modulates the Met receptor and or HGF activity compared to a control without the agonist or antagonists and or selecting the antagonist or agonist that binds to the Met receptor and or HGF . Activities of HGF and or the Met receptor include phosphorylation of Met stimulation of cell proliferation and stimulation of cell migration.

A compound that is identified or designed as a result of any of these methods can be obtained or synthesized and tested for its biological activity for example binding to HGF and or Met receptor and or modulation of HGF HGF and or Met activity. Other modulators of the Met receptor include for example monoclonal antibodies directed against the Met receptor peptide s that can modulate Met receptor function or small molecule compounds such as organic and inorganic molecules which can be identified with methods of the present disclosure.

Transformation of the structure coordinates for all or a portion of Met or the HGF Met complex or one of its ligand binding sites for structurally homologous molecules as defined below or for the structural equivalents of any of these molecules or molecular complexes as defined above into three dimensional graphical representations of the molecule or complex can be conveniently achieved through the use of commercially available software.

The disclosure thus further provides a machine readable storage medium including a data storage material encoded with machine readable data wherein a machine programmed with instructions for using said data displays a graphical three dimensional representation of any of the molecule or molecular complexes of this disclosure that have been described above. In a preferred embodiment the machine readable data storage medium includes a data storage material encoded with machine readable data wherein a machine programmed with instructions for using the abovementioned data displays a graphical three dimensional representation of a molecule or molecular complex including all or any parts of a Met Met ligand binding site or Met like ligand binding site HGF HGF binding site or HGF Met complex as defined above. In another preferred embodiment the machine readable data storage medium includes a data storage material encoded with machine readable data wherein a machine programmed with instructions for using the data displays a graphical three dimensional representation of a molecule or molecular complex a root mean square deviation from the atoms of the amino acids of not more than 0.05 .

In an alternative embodiment the machine readable data storage medium includes a data storage material encoded with a first set of machine readable data which includes the Fourier transform of structure coordinates and wherein a machine programmed with instructions for using the data is combined with a second set of machine readable data including the X ray diffraction pattern of a molecule or molecular complex to determine at least a portion of the structure coordinates corresponding to the second set of machine readable data.

For example a system for reading a data storage medium may include a computer including a central processing unit CPU a working memory which may be for example RAM random access memory or core memory mass storage memory such as one or more disk drives or CD ROM drives one or more display devices e.g. cathode ray tube CRT displays light emitting diode LED displays liquid crystal displays LCDs electroluminescent displays vacuum fluorescent displays field emission displays FEDs plasma displays projection panels etc. one or more user input devices e.g. keyboards microphones mice track balls touch pads etc. one or more input lines and one or more output lines all of which are interconnected by a conventional bidirectional system bus. The system may be a stand alone computer or may be networked e.g. through local area networks wide area networks intranets extranets or the internet to other systems e.g. computers hosts servers etc. . The system may also include additional computer controlled devices such as consumer electronics and appliances.

Input hardware may be coupled to the computer by input lines and may be implemented in a variety of ways. Machine readable data of this disclosure may be inputted via the use of a modem or modems connected by a telephone line or dedicated data line. Alternatively or additionally the input hardware may include CD ROM drives or disk drives. In conjunction with a display terminal a keyboard may also be used as an input device.

Output hardware may be coupled to the computer by output lines and may similarly be implemented by conventional devices. By way of example the output hardware may include a display device for displaying a graphical representation of a binding site of this disclosure using a program such as QUANTA as described herein. Output hardware might also include a printer so that hard copy output may be produced or a disk drive to store system output for later use.

In operation a CPU coordinates the use of the various input and output devices coordinates data accesses from mass storage devices accesses to and from working memory and determines the sequence of data processing steps. A number of programs may be used to process the machine readable data of this disclosure. Such programs are discussed in reference to the computational methods of drug discovery as described herein. References to components of the hardware system are included as appropriate throughout the following description of the data storage medium.

Machine readable storage devices useful in the present disclosure include but are not limited to magnetic devices electrical devices optical devices and combinations thereof. Examples of such data storage devices include but are not limited to hard disk devices CD devices digital video disk devices floppy disk devices removable hard disk devices magneto optic disk devices magnetic tape devices flash memory devices bubble memory devices holographic storage devices and any other mass storage peripheral device. It should be understood that these storage devices include necessary hardware e.g. drives controllers power supplies etc. as well as any necessary media e.g. disks flash cards etc. to enable the storage of data.

HGF Met and or HGF Met complex modulator compounds obtained by methods of the invention are useful in a variety of therapeutic settings. For example Met antagonists designed or identified using the crystal structure of an extracellular fragment of Met or HGF Met complex can be used to treat disorders or conditions where inhibition or prevention of Met binding or activity is indicated.

Likewise Met agonists designed or identified using the crystal structure of the HGF Met complex or an extracellular fragment of Met can be used to treat disorders or conditions where induction or stimulation of Met activity is indicated.

An indication can be for example inhibition or stimulation of Met phosphorylation and the concomitant activation of a complex set of intracellular pathways that lead to cell growth differentiation and migration in a variety of cell types. The ability of HGF to stimulate mitogenesis cell motility and matrix invasion gives it and the Met receptor a central role in angiogenensis tumorogenesis and tissue regeneration. Another indication can be for example in inhibition or stimulation of embryonic development. Still another indication can be for example in inhibition or stimulation of tissue regeneration. Another indication can be for example in inhibition of angiogenesis mitogenesis and or vasculogenesis. Expression of HGF has been associated with thyroid cancer colon cancer lymphoma prostate cancer and multiple myeloma. Yet another indication can be for example in inhibition or stimulation of the HGF Met signaling pathway. Still yet another indication can be for example in inhibition of invasive tumor growth and metastasis.

Met and or HGF Met complex antagonists are also useful as chemosensitizing agents useful in combination with other chemotherapeutic drugs in particular drugs that induce apoptosis. Examples of other chemotherapeutic drugs that can be used in combination with chemosensitizing Met inhibitors include topoisomerase I inhibitors e.g. camptothesin or topotecan topoisomerase II inhibitors e.g. daunomycin and etoposide alkylating agents e.g. cyclophosphamide melphalan and BCNU tubulin directed agents e.g. taxol and vinblastine and biological agents e.g. antibodies such as anti CD20 antibody IDEC 8 immunotoxins and cytokines . Examples of other chemotherapeutic drugs that can be used in combination with chemosensitizing HGF inhibitors include topoisomerase I inhibitors e.g. camptothecin or topotecan topoisomerase II inhibitors e.g. daunomycin and etoposide alkylating agents e.g. cyclophosphamide melphalan and BCNU tubulin directed agents e.g. taxol and vinblastine and biological agents e.g. antibodies such as anti CD20 antibody IDEC 8 anti VEGF antibody immunotoxins and cytokines . Other examples of chemotherapeutic agents include alkylating agents such as thiotepa and CYTOXAN cyclosphosphamide alkyl sulfonates such as busulfan improsulfan and piposulfan aziridines such as benzodopa carboquone meturedopa and uredopa ethylenimines and methylamelamines including altretamine triethylenemelamine trietylenephosphoramide triethiylenethiophosphoramide and trimethylolomelamine acetogenins especially bullatacin and bullatacinone a camptothecin including the synthetic analogue topotecan bryostatin callystatin CC 1065 including its adozelesin carzelesin and bizelesin synthetic analogues cryptophycins particularly cryptophycin I and cryptophycin 8 dolastatin duocarmycin including the synthetic analogues KW 2189 and CB1 TM1 eleutherobin pancratistatin a sarcodictyin spongistatin nitrogen mustards such as chlorambucil chlornaphazine cholophosphamide estramustine ifosfamide mechlorethamine mechlorethamine oxide hydrochloride melphalan novembichin phenesterine prednimustine trofosfamide uracil mustard nitrosureas such as carmustine chlorozotocin fotemustine lomustine nimustine and ranimnustine antibiotics such as the enediyne antibiotics e.g. calicheamicin especially calicheamicin gamma1I and calicheamicin omegaI1 see e.g. Agnew 33 183 186 1994 dynemicin including dynemicin A bisphosphonates such as clodronate an esperamicin as well as neocarzinostatin chromophore and related chromoprotein enediyne antibiotic chromophores aclacinomysins actinomycin authramycin azaserine bleomycins cactinomycin carabicin caminomycin carzinophilin chromomycinis dactinomycin daunorubicin detorubicin 6 diazo 5 oxo L norleucine ADRIAMYCIN doxorubicin including morpholino doxorubicin cyanomorpholino doxorubicin 2 pyrrolino doxorubicin and deoxydoxorubicin epirubicin esorubic in idarubicin marcellomyc in mitomycins such as mitomycin C mycophenolic acid nogalamycin olivomycins peplomycin potfiromycin puromycin quelamycin rodorubicin streptonigrin streptozocin tubercidin ubenimex zinostatin zorubicin anti metabolites such as methotrexate and 5 fluorouracil 5 FU folic acid analogues such as denopterin methotrexate pteropterin trimetrexate purine analogs such as fludarabine 6 mercaptopurine thiamiprine thioguanine pyrimidine analogs such as ancitabine azacitidine 6 azauridine carnofur cytarabine dideoxyuridine doxifluridine enocitabine floxuridine androgens such as calusterone dromostanolone propionate epitiostanol mepitiostane testolactone anti adrenals such as aminoglutethimide mitotane trilostane folic acid replenisher such as frolinic acid aceglatone aldophosphamide glycoside aminolevulinic acid eniluracil amsacrine bestrabucil bisantrene edatraxate defofamine demecolcine diaziquone elformithine elliptinium acetate an epothilone etoglucid gallium nitrate hydroxyurea lentinan lonidainine maytansinoids such as maytansine and ansamitocins mitoguazone mitoxantrone mopidanmol nitraerine pentostatin phenamet pirarubicin losoxantrone podophyllinic acid 2 ethylhydrazide procarbazine PSK polysaccharide complex JHS Natural Products Eugene Oreg. razoxane rhizoxin sizofuran spirogermanium tenuazonic acid triaziquone 2 2 2 trichlorotriethylamine trichothecenes especially T 2 toxin verracurin A roridin A and anguidine urethan vindesine dacarbazine mannomustine mitobronitol mitolactol pipobroman gacytosine arabinoside Ara C cyclophosphamide thiotepa taxoids e.g. TAXOL paclitaxel Bristol Myers Squibb Oncology Princeton N.J. ABRAXANE Cremophor free albumin engineered nanoparticle formulation of paclitaxel American Pharmaceutical Partners Schaumberg Ill. and TAXOTERE doxetaxel Rhone Poulenc Rorer Antony France chloranbucil GEMZAR gemcitabine 6 thioguanine mercaptopurine methotrexate platinum analogs such as cisplatin and carboplatin vinblastine platinum etoposide VP 16 ifosfamide mitoxantrone vincristine NAVELBINE vinorelbine novantrone teniposide edatrexate daunomycin aminopterin xeloda ibandronate CPT 11 topoisomerase inhibitor RFS 2000 difluoromethylornithine DMFO retinoids such as retinoic acid capecitabine and pharmaceutically acceptable salts acids or derivatives of any of the above.

Also included in the definition of chemotherapeutic agent above are anti hormonal agents that act to regulate or inhibit hormone action on tumors such as anti estrogens and selective estrogen receptor modulators SERMs including for example tamoxifen including NOLVADEX tamoxifen raloxifene droloxifene 4 hydroxytamoxifen trioxifene keoxifene LY117018 onapristone and FARESTON toremifene aromatase inhibitors that inhibit the enzyme aromatase which regulates estrogen production in the adrenal glands such as for example 4 5 imidazoles aminoglutethimide MEGASE megestrol acetate AROMASIN exemestane formestanie fadrozole RIVISOR vorozole FEMARA letrozole and ARIMIDEX anastrozole and anti androgens such as flutamide nilutamide bicalutamide leuprolide and goserelin as well as troxacitabine a 1 3 dioxolane nucleoside cytosine analog antisense oligonucleotides particularly those which inhibit expression of genes in signaling pathways implicated in abherant cell proliferation such as for example PKC alpha Ralf and H Ras ribozymes such as a VEGF expression inhibitor e.g. ANGIOZYME ribozyme and a HER2 expression inhibitor vaccines such as gene therapy vaccines for example ALLOVECTIN vaccine LEUVECTIN vaccine and VAXID vaccine PROLEUKIN rIL 2 LURTOTECAN topoisomerase 1 inhibitor ABARELIX rmRH and pharmaceutically acceptable salts acids or derivatives of any of the above.

The HGF chain or variants thereof form cocrystals with the Met receptor in accord with the methods described herein. The crystals also are useful to store and or deliver HGF chain or Met receptor molecules. HGF is useful as an inhibitor of full length HGF and can be used to identify or design other like inhibitors. HGF Met complex may be useful as a standards in assays including diagnostic assays. Crystals can be prepared and used to store HGF chain Met receptor complexes for later use.

A variety of methods are known to those of skill in the art for formation of crystals. In some embodiments for crystals prepared for storage the crystal size and structure does not have to be so uniform or homogenous as for X ray diffraction. In other embodiments the crystals effectively diffract x rays to a resolution of 5 or better. Typically a purified polypeptide is contacted with a precipitant in the presence of a buffer. Precipitants include salts polymers or organic molecules. Organic precipitants include isopropanol ethanol hexanediol and 2 methyl 2 4 pentanediol. Polymeric precipitants include polyethylene glycol and polyamines. Salts used include ammonium sulfate sodium citrate sodium acetate ammonium dichloride sodium chloride and magnesium formate. Many buffers can be utilized and are known to those of skill in the art.

In some cases crystals can be cross linked to one another. Such cross linking may enhance the stability of the crystal. Methods of cross linking crystals are know to those of skill in the art and have been described for example in U.S. Pat. No. 5 849 296.

The crystals can be maintained in crystallization solution they can be dried or combined with other carriers and or other ingredients to form compositions and formulations. In some embodiments the crystals can be combined with a polymeric carrier for stability and sustained release. Formulations of crystals of proteins such as enzymes receptors antibodies and like molecules or fragments thereof can include at least one ingredient or excipient. Ingredient or expedients are known to those of skill in the art and include acidifying agents aerosol propellants alcohol denaturants alkalizing agents anti caking agents antifoaming agents microbial preservatives anti antioxidants buffering agents lubricants chelating agents colors desiccants emulsifying agents filtering aids flavors and perfumes humectants ointments plasticizers solvents e.g. oils or organic sorbents carbon dioxide sorbents stiffening agents suppository bases suspending or viscosity increasing agents sweetening agents tablet binders table or capsule diluents tablet disintegrants tablet or capsule lubricants tonicity agent flavored or sweetened vehicles oleaginous vehicles solid carrier vehicles water repelling agent and wetting or solubilizing agents.

In some embodiments the ingredients enhance storage stability. In other embodiments the ingredient or excipient is preferably selected from the group consisting of albumin sucrose trehalose lactitol gelatin and hydroxypropyl cyclodextran.

All publications patents and patent documents are incorporated by reference herein as though individually incorporated by reference. The disclosure has been described with reference to various specific and preferred embodiments and techniques. However it should be understood that many variations and modifications can be made while remaining within the spirit and scope of the disclosure.

For crystallization we used a construct of the Met receptor that contained residues 25 to 567 and an additional C terminal His tag. In addition the cleavage site after residue Arg308 present in the Met receptor family but not in the Sema domain of the Semaphorins or plexins was replaced with a thrombin cleavable site. This exchange residues 304 to 308 were exchanged from KRKKR SEQ ID NO 4 to LVPRG SEQ ID NO 5 was introduced since native Met expressed in insect cells was partially cleaved and not homogeneous. Mutation of the cleavage site allowed for the production of homogeneous protein that depending on treatment with thrombin was cleaved or uncleaved. The complex presented here was obtained with the unprocessed single chain Met.

The HGF chain protein used for crystallization begins with residue Val495 the N terminal residue obtained after maturation of single chain HGF and has one alteration from the natural sequence in full length mature HGF the and chains are covalently tethered by a disulfide bridge putatively formed between Cys487 and Cys604 Donate et al. 1994 . In order to prevent artificial dimerization of HGF chains during purification and crystallization Cys604 was replaced with serine. SEQ ID NO 1 The complex of the HGF chain and the Met fragment was formed in solution purified using size exclusion chromatography and set up for crystallization trials.

The cDNA for the full length Met receptor was kindly provided by D. Wickramasinghe. PCR was carried out on the region encoding residues 25 567 of the Met receptor. A 2 step overlapping PCR scheme was used to introduce the thrombin cleavage site LVPRG SEQ ID NO 5 and remove the native sequence KRKKR SEQ ID NO 4 . An N terminal primer added overlapping sequence with the C terminus of the insect cell secretion signal present in the pAcGP67A vector BD Biosciences . A C terminal primer contained coding sequence for His and a NotI restriction site added directly onto the C terminus. For placement into the pAcGP67A vector an additional PCR was carried out on the coding region for the insect cell secretion signal with additional overlapping sequence from the N terminus of our Met construct. A final PCR was done to directly fuse the C terminus of the secretion signal to the N terminus of our Met construct. This PCR product as well as pAcGP67A were digested with SpeI and NotI. Insert was combined with vector at a 3 1 ratio ligated and transformed into XL 1 Blue cells Stratagene .

Purified plasmid DNA pAcGP67A plus Met Sema and PSI domain was transfected into Sf9 insect cells according to the manufacturer s protocol BD Biosciences . Viral stock was amplified 3 times before use in protein expression. For expression 1 L of Hi5 insect cells growing in ESF 921 media Protein Expression LLC at a density of 5 10cells mL were infected with 10 mL of viral stock from the 3amplification and incubated at 27 C. for 72 hours. Cells were then removed from the supernatant by centrifugation at 3000 g for 15 minutes. 1 mM of NiCl 5 mM of CaCl and 50 mM Tris pH 8.0 were added to the supernatant. The supernatant was then filtered through a 0.2 m vacuum filter. Supernatant was applied to a 2 mL Ni NTA column Qiagen by gravity flow followed by 20 mL of wash buffer 50 mM Tris pH 8.0 500 mM NaCl 5 mM imidazole . Protein was eluted with a buffer containing 50 mM Tris 8.0 300 mM NaCl 250 mM imidazole. Fractions containing Met were identified by SDS PAGE analysis and pooled concentrated to 500 L and loaded onto a Superdex S 75 gel filtration column Amersham Biosciences equilibrated with 50 mM Tris 8.0 500 mM NaCl. The final yield for Met Sema and PSI domain was approximately 0.25 mg L of insect cell supernatant.

The amino acid sequence of an extracellular fragment of Met is shown in Table 6. SEQ ID NO 3 . Another embodiment of an extracellular fragment of Met comprises an amino acid sequence of SEQ ID NO 10 and is shown in .

HGF proteins were expressed in insect cells using baculovirus secretion vector pAcGP67 Pharmingen San Diego Calif. . All constructs contained a Histag at the carboxy terminus and were purified to homogeneity 95 purity by Ni NTA metal chelate and gel filtration chromatography. For wildtype HGF a cDNA fragment encoding the HGF chain from residues Val495 c16 to Ser728 c250 was cloned by PCR such that Val495 c16 was inserted immediately after the secretion signal sequence. Site directed mutagenesis was carried out using QuikChange Stratagene La Jolla Calif. with oligonucleotide 5 CCTAATTATGGATCCACAATTCCTG3 SEQ ID NO 6 to make HGF containing a Cys604 to Ser mutation HGF . The cysteine was not altered in this construct to allow putative disulfide formation between Cys487 and Cys604. Wildtype HGF SEQ ID NO 14 refers to the native sequence and HGF SEQ ID NO 1 refers to the C604S mutant. Numbering is as follows full length HGF sequence starting with MWV . . . as numbers 1 3 chymotrypsinogen numbering is shown in the brackets . It will be readily apparent that the numbering of amino acids in other isoforms of HGF may be different than that of the HGF numbering disclosed herein. The disclosure provides sequential numbering based on sequence only. In embodiments an isoform may have structural differences for example if it carries insertion s or deletion s relative to the HGF reference sequence. The chymotrypsinogen numbering convention may be useful for comparison to workers in the field.

Baculovirus vectors containing the desired inserts were transfected into Sf 9 cells on plates in TNM FH media via the Baculogold Expression System according to manufacturer s instructions Pharmingen San Diego Calif. . After 2 4 rounds of virus amplification 10 mL of viral stock was used to infect 1 L of High Five cells Invitrogen San Diego Calif. in suspension at 5 10cells mL in TNM FH media. Cultures were incubated at 27 C. for 72 h before harvesting the culture media by centrifugation at 8 000 g for 15 min. Cell culture media was applied to a 4 mL Ni NTA agarose column Qiagen Valencia Calif. . After washing with 4 column volumes of 50 mM Tris.HCl pH 8.0 500 mM NaCl 5 mM imidazole HGF proteins were eluted with 50 mM Tris.HCl pH 8.0 500 mM NaCl 500 mM imidazole. The eluate was pooled and applied to a Superdex 200 column Amersham Biosciences Piscataway N.J. equilibrated in 10 mM HEPES pH 7.2 150 mM NaCl 5 mM CaCl. Protein peaks were collected and concentrated using a Centriprep YM 10 Millipore Bedford Mass. . Fractions were analyzed by 12 SDS PAGE stained with Coomassie blue.

Met was combined with HGF chain in a 1 1.5 ratio and concentrated for loading onto a Superdex S 200 gel filtration column Amersham Biosciences equilibrated with 10 mM Tris 8.0 250 mM NaCl. Elution fractions were assayed with SDS PAGE and those containing the complex were pooled. Buffer exchange into 10 mM Tris 8.0 125 mM NaCl was carried out and the complex was then concentrated to 10 mg mL for crystallization trials. Although crystals grew without deglycosylation diffraction was enhanced after deglycosylating the Met protein. For this Met previously purified using Nickel chromatography as described above was mixed with a combination of PNGase F Roche and Endoglycosidase A Roche at 1 100 enzyme Met concentration and incubated at room temperature for 3 hours. Met was then combined with HGF chain concentrated and loaded onto a gel filtration column as described above for the non deglycosylated material.

Met HGF chain crystallized at 19 C. in 4 L hanging drops in a 1 1 ratio of protein to mother liquor suspended over mother liquor 12 15 PEG 10 000 Tris pH 8.0 . Crystals grew to approximately 100 m 200 m 200 m in 3 to 7 days. For data collection crystals were placed in a solution of mother liquor containing 25 xylitol for 30 seconds and then flash frozen in liquid N.

Data were collected at CHESS beamline A1 to 3.2 resolution and processed using the HKL package Otwinowski et al. 1997 . The crystals belonged to space group P222 with cell parameters of a 137.1 b 186.4 and c 66.7 and contained 1 complex of Met HGF chain in the asymmetric unit and thus have a solvent content of about 75 . The structure was solved by molecular replacement with the program AMoRe Navaza 1994 with the crystal structure of HGF chain alone structural coordinates available at RCS Protein Databank Accession No PDB1UX3 as a search model. Using data from 8 to 4 yielded a clear solution in the rotation and translation functions and the best solution was subjected to rigid body refinement in program Refmac CCP4 1994 . Solvent flattening which due to the high crystal solvent content led to dramatic improvements in the quality of the phases yielded maps that were of sufficient quality to recognize the propeller of the Sema domain and allowed for building the Met receptor without the use of any additional models. Subsequent rounds of model building with program 0 Jones et al. 1991 and further refinement with program Refmac resulted in the final model with an Rfree and Rcryst of 27.0 and 20.9 respectively for all data between 30 and 3.2 .

The crystal structure of HGF Met was analyzed to identify amino acid positions involved in binding and or activation of the Met receptor by HGF .

The chain of HGF shares close to 40 sequence identity with the protease domain of plasmin a trypsin like serine protease. Structurally these enzymes can be described as globular proteins composed of two antiparallel barrel domains . Both of the barrels share the same general topology and are formed by six antiparallel strands with the N terminal four strands folding into a Greek key motif followed by two strands that form a hairpin. HGF like serine proteases is expressed as a zymogen like precursor. For serine proteases structural studies have shown that insertion of the N terminus resulting from the maturation process leads to allosteric rearrangments within the binding site for the protease substrate Perona and Craig 1995 Hedstrom 2002 . In serine proteases the newly formed N terminus forms a salt bridge with a nearby aspartate which leads to the formation of the oxyanion hole via the backbone NH s of two neighboring residues and the creation of a part of the S1 pocket . The activated form of the HGF chain as seen in the structure presented here is very similar to the mature form of serine proteases superposition with plasmin pdb code 1BML yields an rmsd of about 1.3 for 212 C pairs. Superposition between HGF chain and plasminogen pdb code 1QRZ reveal differences in their activation domain Freer et al. 1970 and only 198 C pairs can be aligned well.

As with serine proteases the N terminal amine of Val495 is inserted into the core of the C terminal barrel and forms a salt bridge with the carboxyl group of Asp672 which corresponds to Asp194 in plasmin presumably leading to similar rearrangements which have been characterized in serine proteases. The region in HGF corresponding to this so called activation domain i.e. the loops that undergo conformational changes in serine proteases Freer et al. 1970 Huber and Bode 1978 forms part of the binding surface with Met discussed below. Therefore proper display of the residues that interact with Met requires maturation of HGF.

After maturation the and chains of HGF remain connected via a disulfide bond. Based on alignments of the kringle domains K1 to K4 of HGF and macrophage stimulating protein MSP the two cysteine residues responsible for the formation of this disulfide bond were identified as Cys487 on the chain of HGF and Cys604 on the chain Donate et al. 1994 . The crystal structure however raises the possibility of an alternative cysteine residue as the anchor for the HGF chain. The HGF chain has two 2 cysteine residues that do not have partners for the formation of disulfide bridges within the chain. Neither is conserved in plasmin or MSP and both are located on the periphery of the protein and are at least partially solvent exposed either could be potential partners for the formation of a disulfide bond with the chain while the other remains unpaired. The superposition of the plasmin structure that contains part of the chain and the HGF chain structure presented here shows that the distance between the C terminus of the plasmin chain and Cys604 of the HGF chain is 27 and thus only slightly shorter than the respective distance between the plasmin chain C terminus and Cys561 33 . In addition the sequence alignment between plasmin MSP and HGF shows that the chain cysteine forming the disulfide bond in plasmin and MSP is 13 and 15 residues away from the cleavage site in those proteins respectively but there are only 7 residues in HGF to span the distance from Cys487 to the cleavage site. Therefore due to distance requirements the C terminus of the HGF chain cannot follow the same path on the surface of the chain as it does in plasmin or MSP. Continuing work seeks to clarify which of the two free cysteine residues in HGF forms the disulfide bond with the chain of HGF.

The Sema domain of Met forms 7 bladed propeller with a diameter of a little more than 50 . The overall shape of the domain resembles a funnel with an inner diameter of about 25 between main chain atoms at the wide portion and 10 in the narrowest part . Generally in propellers each of the blades is formed by 4 antiparallel strands with strand A in the center of the propeller followed by strands B and C and with strand D forming the outermost strand of the blade. The blades are arranged in a circular fashion with the N terminal strand forming strand D of the last blade thus closing the propeller and stabilizing the overall structure . The AB and CD loops of each blade of the Met Sema domain form the relatively flat bottom face and the generally longer BC and DA loops form the top face of the propeller . In Met the position of the 6th and the 7th blade are off center with blade 7 being closer to the center of the barrel and blade 6 more distant. This gives the domain an overall oval shape.

The propeller of the Met Sema domain is structurally most closely related to the recently reported crystal structures of Sema4D Love et al. 2003 and Sema3A Antipenkov et al. 2003 . With the exception of the D strands in blades 3 and 5 the core of Sema4D and the Met Sema domain align well and the superposition of residues 40 to 519 with the Sema domain of Sema4D results in an mmsd of 1.6 for 303 atom pairs . The residues that form strand D5 in Sema4A are hydrogen bonding to strand D4 in Met thus the 4th blade of Met contains an extra strand while blade 5 is missing its strand D. While the core of both proteins align well the loops contain a number of insertions or deletions and generally adopt very different conformations. For example the Sema domains of Met and of Sema4D both have an insert of about 20 residues following strand D1. In Met these residues form a short antiparallel two stranded sheet while Sema4D has an helix . Of the 24 loops that connect the various strands only 3 have no insertions or deletions when comparing Met with Sema4D or Sema3A . Interestingly both the Semaphorins and the Met receptor have their largest insertion between strands C5 and A6 however these inserts are not related in structure or sequence to each other. In Sema3A this insert is involved in dimer formation and has been shown to be important for neuropilin binding Antipenko et al. 2003 . In the Met Sema domain some of the residues within the insert are disordered and not included in our model see . The function of the insert in Met is not known but its proximity to the HGF chain binding site discussed below suggests that it might be involved in binding to the chain of HGF.

The loop containing the cleavage site between the and chain of Met connects strand D4 to A5 it is disordered in the electron density. After processing of native Met the chain which forms the N terminal 4 blades of the Sema domain and the chain remain connected via at least 2 disulfide bonds. One disulfide bond is formed between residues Cys298 on strand D4 and Cys363 on C5 and the second is formed between Cys282 D3 and Cys409. This last cysteine is positioned in a disordered region of the insert after blade 4. The electron density does not allow tracing of the main chain in this area but there is sufficient density in the area of the S position to suggest that the disulfide bond is at least partially formed. With the exception of the N terminal Cys26 which is disordered in the presently disclosed crystal structure and has no potential partner in the construct that was used for crystallization all cysteine residues are engaged in disulfide bonds. It is unclear if Cys26 remains unpaired in the context of full length Met receptor or if it forms a disulfide bond with another cysteine that resides in the IPT domains. Besides the two disulfide bonds that covalently connect the and chain of the Sema domain the domain is stabilized by 5 additional disulfide bridges formed between Cys95 Cys101 Cys98 Cys159 Cys133 Cys141 Cys172 Cys175 and Cys385 Cys397 .

The surface of the Met Sema domain appears to be mostly negatively charged especially in the center of the bottom face of the Sema domain . This surface which is formed by a 20 residue insertion after strand D in the first blade and includes the rather irregular area of strand D in the third blade shows clusters of acidic residues. Monomeric full length Met was shown to bind to heparin Gherardi et al. 2003 . In presently disclosed structure there are no dominant positively charged patches within the Met Sema domain however two arginines are located in the vicinity of the disordered furin cleavage site. Together with the 6 positively charged residues that reside in this loop these arginines could constitute a potential heparin binding site.

The last strand of the Sema domain C7 is followed immediately by the PSI domain. This domain with dimensions of about 20 15 contains four 4 disulfide bridges and is not an integral part of the Sema domain but rather an independent structural module. The small core of the domain is formed by a helix and a short two stranded antiparallel i sheet that are connected via a disulfide bond and sandwich the side chain of Trp540. Superposition of this domain with the PSI domain of Sema4D results in an rmsd of 1.6 for 41 C pairs. The relative orientation between the Sema and the PSI domains in the Met and the Sema4D structures however is different and requires a rotation of about 40 for superposition.

The HGF chain binds to the Sema domain of Met at the bottom face of the propeller and forms contacts with residues that protrude from blades 2 and 3 . This is unlike other propeller containing receptors such as the integrins which bind their ligands utilizing the top face of the propeller Xiong et al. 2002 . Other Sema domain containing proteins such as Sema3A and Sema4D are also postulated to bind ligands using the top face Antipenko et al. 2003 Love et al. 2003 .

The interface between HGF chain and Met buries a total of 1770 of solvent accessible surface and is dominated by polar or charge charge interactions. Of the 20 HGF residues that have more that 10 buried in the interface 6 are positively charged 3 carry negative charges and 3 are aromatic. Even more dramatic of the 18 residues contributing more than 10 of buried surface to the interface on the Met side 6 carry negative charges 4 are positively charged and 3 are aromatic. The combined buried surface of the hydrophobic residues Ala Val Leu Ile Met and Phe amounts to a mere 145 while the charged residues Asp Glu Lys Arg and His are responsible for more than 1 000 of buried surface. The resolution of the presently disclosed structure does not permit a detailed discussion of hydrogen bonds however the position of the side chains suggest the presence of at least 7 charged interactions between Met and the HGF chain. See Table 3 

On HGF the residues analogous to those responsible for substrate binding and catalysis in the related serine proteases form most of the interactions with Met. Serine proteases bind linear peptides to their recognition site and form hydrogen bonds to main chain atoms to position a specific peptide bond for proteolytic cleavage. In contrast the HGF chain binds to a series of protruding polar side chains from Met that originate mainly from 3 separate loops . The first of these loops includes residues 124 128 and connects strands A2 and B2 of Met. This loop contains two tyrosines that are in the core of the binding interface and pack against Arg695 of HGF. The second loop of Met that contacts HGF chain contains residues Asp190 Arg191 and Phe192 and connects A3 and B3. Asp190 of Met forms charged interactions with Arg533 of HGF. The side chain of Arg191 of Met packs against Val692 and Pro693 of HGF with its hydrophobic portion and in the presently disclosed model is in hydrogen bonding distance to the side chains of Glu656 and Asp578 with its guanidinium group. This Asp578 corresponds to aspartic acid c102 in the catalytic triad of the related serine proteases. Furthermore the backbone of this loop forms hydrophilic interactions with Gln534 the residue homologous to the histidine c57 of the protease catalytic triad.

The third segment contributing to the interface also includes strand D in the third blade of the propeller an area that deviates from the classical propeller fold . Strand D is interrupted and has a short helical insert within its strand. This unusual conformation presents a number of residues towards the surface of the HGF chain. The most prominent interaction formed by this loop involves Glu221 which extends its side chain towards what would be the S1 binding site in serine proteases . The carboxylate of Glu221 forms an extensive network of polar interactions with the side chain of Tyr673 corresponding to the serine c195 of the catalytic triad in proteases as well as the backbone amides of Gly694 and Gly696 however unlike in serine proteases where the S1 pocket is filled by the specificity determining P1 residue it does not penetrate deeply into the S1 pocket. As a result of the exchange of Ser to Tyr in position 673 c195 the entrance of the S1 pocket is much smaller than in the related proteases. Thus Glu221 projects towards the entrance of the S1 pocket from a different angle when compared to a complex between trypsin and bovine pancreatic trypsin inhibitor BPTI and does not fill the pocket . Therefore a large cavity presumably filled with a number of water molecules remains in HGF chain.

The structures of the HGF chain bound to Met and in its unbound state as disclosed in application U.S. Ser. No. 60 569 301 filed May 6 2004 which application is hereby incorporated by reference are very similar and superimpose with an rmsd of less than 0.8 for 218 C pairs. The only significant differences between the two structures involve residues that are close to the Met binding interface. The backbone of residues 645 to 651 is disordered in unliganded HGF but well defined in the structure of the complex where it adopts a conformation that is commonly observed in serine proteases . The side chains of the basic residues Arg647 and Lys649 are not well defined in the electron density but project towards the Met binding interface and in presently disclosed model participate in the formation of the complex. Arg695 also undergoes a change in conformation upon Met binding. This residue located in the center of the receptor binding interface packs against Tyr125 of Met and its backbone forms part of the S1 pocket. In the structure of unliganded HGF the conformation of the backbone in this area is different and the side chain of Arg695 projects toward the disordered region of the structure. It is conceivable that the binding event alters the position of Arg695 which in turn allows residues 645 to 651 to adopt a more stable conformation.

In embodiments a model for HGF mediated Met receptor activation is provided. Little is known about the signaling complex itself and details of this interaction such as the stoichiometry between ligand and receptor have been elusive.

The presently disclosed structure shows unambiguously how Met binds to the chain of HGF. However the exact role this interaction might play in Met activation remains uncertain. Two general models of how HGF can lead to Met dimerization have been suggested Miller and Leonard 1998 . One of these models follows the growth hormone paradigm de Vos et al 1994 where one HGF molecule binds to two different Met receptors with different affinities to form the 2 1 signaling complex. The other model suggests that two 1 1 complexes of Met and HGF are dimerized either via heparin or an alternative mechanism to form stable 2 2 complexes.

In the present disclosure complexes of HGF chain and the Sema domain of Met were purified from the individual components via size exclusion chromatography. This clearly showed that HGF and the N terminal 564 residues of Met form a stable 1 1 complex. If Met activation via HGF indeed entails a 2 1 complex the present disclosure characterizes the low affinity binding interaction between HGF and Met. The high affinity binding interaction involves the chain of HGF in particular the NK1 domains Lokker et al. 1992 and a previously uncharacterized binding site on Met. This model is in good agreement with the notion that single chain HGF is able to bind Met via its high affinity binding site which is contained in its N terminal portion yet unable to signal. For signaling maturation of HGF is required. This leads to rearrangements in the activation domain of HGF the formation of the low affinity binding site and possibly allows for a reorientation of the chain of HGF relative to its chain. This model could also be valid for the closely related Ron MSP system. Like HGF single chain MSP can bind to its receptor Ron but is only able to signal in its matured 2 chain form. Interestingly and in contrast to HGF the high affinity binding site in MSP is contained within its chain while the low affinity binding site is harbored by its chain Danilkovitch et al. 1999 .

Recent studies employing analytical ultracentrifugation and light scattering experiments indicate that mature HGF and the Met ectodomain form 1 1 complexes that can be stabilized with heparin to form a 1 1 complex Gherardi et al. 2003 . However in those experiments no higher order complexes or aggregates could be detected.

The present disclosure shows that the low affinity interaction between HGF and Met is sufficient for 1 1 complex formation. The absence of 2 1 complexes in Gherardi et al 2003 suggest two possible scenarios. Firstly it is possible that the high and low affinity binding sites on the receptor overlap. In this case at a 1 1 molar ratio of ligand and receptor all high affinity binding sites on the receptor are occupied and none of the low affinity binding sites available thus preventing the formation of 2 1 complexes. An alternative explanation for the absence of 2 1 complexes is that and chain of a single HGF molecule bind to the same Met receptor and form stable 1 1 complexes. These 1 1 complexes only associate very weakly and require the juxtamembrane or intracellular portion of the receptors or need additional molecules or co receptors for the assembly of 2 2 or higher order signaling complexes.

Recent crystal structures of dimeric Sema4D and Sema3A have demonstrated how these Sema domains dimerize. Antipenko A Himanen J P van Leyen K Nardi Dei V Lesniak J Barton W A Rajashankar Kr Lu M Hoemme C Puschel A W Nikolov D B 2003 Structure of the Semaphorin 3A receptor binding module. 39 589 598 and Love C A Harlos K Mavaddat N Davis S J Stuart D I Jones E Y Esnouf R M 2003 . The ligand binding face of the Semaphorins revealed by the high resolution crystal structure of Sema4D. 10 843 848.

In Sema4D dimerization of the Sema domains is mediated by four loops that protrude from the core of the propeller. Three of these loops connect strands B4 C4 D4 A5 and B5 C5 and the fourth is part of the insert that follows the 5th blade of the propeller . In the present crystal structure the interface observed in the Semaphorin structures is not present. The superposition of the complex described here and the dimer of Sema4D shows that none of the regions relevant for dimerization in Sema4D are conserved in the Met structure . Thus it is unlikely that the Met dimerization follows the Semaphorin paradigm. However since the loop connecting D4 A5 contains the furin cleavage site that was altered in the present construct the possibility that the dimer seen in Sema4D is similar in the Met receptor system cannot be excluded. A number of characteristics of Met speak against this model. As discussed above cleavage of Met at the furin cleavage site is not required for signaling. Therefore it seems unlikely that this loop is involved in crucial interactions in the signaling complex. Furthermore if Met were to dimerize similarly to the Semaphorins the chains of HGF would be more than 70 apart from each other and lie on opposite sides of the Sema domain. Thus they could not participate in the stabilization of a 2 2 complex yet it is known that maturation of HGF and the presence of the HGF chain are required for the formation of a signaling complex.

A different model for a 2 2 signaling complex is suggested by the crystal packing of the 1 1 complexes in the disclosed crystal structure where two HGF chains form a symmetric interface with about 1250 of total buried surface . This interface is formed via interactions from 3 loops the strand immediately following the N terminus residues 619 630 and residues 662 665. These 3 loops are part of the so called activation domain in the related serine proteases and rearrange upon zymogen maturation Huber and Bode 1978 . As the surface engaged in this interface can only be present if full length HGF is properly cleaved and in its matured form this dimer can only form after maturation which explains the requirement of HGF maturation for proper signaling. In this model the C termini of the Met receptor Sema domains are separated by about 110 a gap that could be easily spanned by the PSI and the 4 IPT domains of the two receptors that connect the Sema domains to the transmembrane helix. Interestingly while HGF chain chain interactions are clearly weak no oligomers were detected during size exclusion chromatography of the complex and the HGF chain is monomeric in solution data not shown the same interface is also present in crystals containing only the HGF chain and that were derived under different crystallization conditions see application U.S. Ser. No. 60 569 301 filed May 6 2004.

Based on these observations we propose a model in which the chain of HGF provides the initial binding event for HGF to Met . The large negatively charged patches in the center of the bottom of the Sema domain propeller may represent a potential binding site for the HGF chain allowing for interaction with the positively charged surface region of the NK1 domain Lokker et al. 1992 Ultsch et al. 1998 . Before cleavage of single chain HGF the protease like domain of HGF would be unavailable for binding to Met. Once cleavage occurs the N terminus of the newly formed chain inserts into the protein and leads to rearrangements in the activation domain creating the Met binding site on HGF chain. Now the HGF chain can bind to Met which positions the chain dimerization surface for contact with a neighboring HGF Met complex. The presence of heparin sulfate proteoglycans may serve to strengthen these interactions.

The 2 2 model is supported by the stoichiometry studies mentioned above which show no dimerization of complexes in solution Gherardi et al. 2003 indicating that the 2 2 complex formation must be very weak. Also the fact that the chain alone NK4 is not able to mediate dimerization but instead acts as an antagonist in vivo Date et al. 1997 further highlights the potential role for the HGF chain in dimerization. The disclosed structure of Met lacks the IPT domains and their role in the signaling of the Met HGF complex is still unclear. A crystal structure of the full length Met in complex with full length activated HGF could further resolve these issues.

Antipenko A Himanen J P van Leyen K Nardi Dei V Lesniak J Barton W A Rajashankar K R Lu M Hoemme C Puschel A W Nikolov D B 2003 39 589 598.

Cioce V Csaky K G Chan A M Bottaro D P Taylor W G Jensen R Aaronson S A Rubin J S 1996 271 13110 13115.

Gherardi E Youles M E Miguel R N Blundell T L Iamele L Gough J Bandyopadhyay A Hartmann G Butler P J 2003 . 100 12039 12044.

Jankowski K Kucia M Wysoczynski M Reca R Zhao D Trzyna E Trent J Peiper S Zembala M Ratajczak J Houghton P Janowska Wieczorek A Ratajczak M Z 2003 63 7926 7935.

Laskowski R A MacArthur M W Moss D S Thornton J M 1993 PROCHECK a program to check the stereochemical quality of protein structures. 26 283 291.

Nakamura T Nishizawa T Hagiya M Seki T Shimonishi M Sugimura A Tashiro K Shimizu S 1989 Nature 342 440 443.

Navaza J 1994 AMoRe an Automated Package for Molecular Replacement. Acta Crystallogr. Sect. A 50 157 163.

Nardone H C Ziober A F LiVolsi V A Mandel S J Baloch Z W Weber R S Mick R Ziober B L 2003 98 1386 1393.

Otwinowski Z Minor W 1997 Processing of X Ray Diffraction Data Collected in Oscillation Mode. Methods Enzymol. 276 307 326.

Winberg M L Noordermeer J N Tamagnone L Comoglio P M Spriggs M K Tessier Lavigne M Goodman C S 1998 . 95 903 916.

